<!--

 Copyright 2016, Institute for Systems Biology

 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

 Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.

-->



<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
    <meta charset="utf-8">
    <meta http-equiv="x-ua-compatible" content="ie=edge">
    <title>ISB-CGC</title>
    <meta name="description" content="ISB-CGC Cancer Gateway in the Cloud">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    
    <link type="text/css" rel="stylesheet" href="/static/css/select2.min.css" />
    <link rel="icon" type="image/x-icon" href="/static/img/favicon.ico" />

    <link type="text/css" rel="stylesheet" href="/static/css/style.css" />
    
    
        <!-- Global site tag (gtag.js) - Google Analytics -->
        <script async src="https://www.googletagmanager.com/gtag/js?id=UA-131643069-1"></script>
        <script>
          window.dataLayer = window.dataLayer || [];
          function gtag(){dataLayer.push(arguments);}
          gtag('js', new Date());

          gtag('config', 'UA-131643069-1');
        </script>
    
    <script>
        var STATIC_FILES_URL = '/static/';

    </script>
</head>
<body id="body">
<!--Include site main header-->
<header class="site-header navbar-fixed-top fixed-top navbar-isb" role="menubar"></header>

<main id="content" class="main-content mt-5 pt-2">
    <div class="page-heading"></div>
    <div class="container-fluid ps-0 pe-0">
        <div class="row">
            <div class="col-lg-12">
                <div id="js-messages"></div>
            </div>
        </div>
    <div>
    <div class="container">
        <div class="row">
            <div id="citations" class="col-lg-12">
                <h2>ISB-CGC Publications</h2>
                <ol>
                    <li>
                        Sheila M. Reynolds, Michael Miller, Phyliss Lee, et al.
                        <a target="_blank" rel="noreferrer"
                           href="https://storage.googleapis.com/isb-cgc-publications/The%20ISB%20Cancer%20Genomics%20Cloud_%20A%20Flexible%20Cloud-Based%20Platform%20for%20Cancer%20Genomics%20Research.pdf">The
                            ISB Cancer Genomics Cloud: A Flexible Cloud-Based Platform for Cancer Genomics Research</a>.
                        Cancer Res, 2017. doi: 10.1158/0008-5472.CAN-17-0617.
                    </li>
                    <li>
                        Kawther Abdilleh, Boris Aguilar, J. Ross Thomson et al.
                        <a target="_blank" rel="noreferrer"
                           href="https://storage.googleapis.com/isb-cgc-publications/Poster-166-ACM2020-Google-ISB-CGC.pdf">Multi-omics
                            data integration in the Cloud: Analysis of Statistically Significant Associations Between
                            Clinical and Molecular Features in Breast Cancer</a>.
                        2020. doi: 10.1145/3388440.3414917.
                    </li>
                    <li>
                        Dondra Bailey, Kawther Abdilleh, Boris Aguilar, et al.
                        <a target="_blank" rel="noreferrer"
                           href="https://storage.googleapis.com/isb-cgc-publications/ISB-CGC-BalieyGriggsLab-ACM-BCB-2020.pdf">Multi-omics
                            characterization of Microtubule-actin cross linking factor 1 (MACF1) using the ISB-Cancer
                            Genomics Cloud</a>.
                        2020. doi: 10.1145/3388440.3414918.
                    </li>
                    <li>
                        Kawther Abdilleh, Boris Aguilar, Ronald C. Taylor, et al.
                        <a target="_blank" rel="noreferrer"
                           href="https://storage.googleapis.com/isb-cgc-publications/ISB-CGC-BiologicalDataScience2020.pdf">Large-scale
                            Cloud-based Inference of Differential Breast Cancer-related Network Gene Between Patient
                            Cohorts</a>.
                        2020.
                    </li>
                    <li>
                        Aguilar B, Gibbs DL, Reiss DJ, et al.
                        <a target="_blank" rel="noreferrer"
                           href="https://storage.googleapis.com/isb-cgc-publications/Aguilar%20et%20al.%20-%202020%20-%20A%20generalizable%20data-driven%20multicellular%20model%20of.pdf">A
                            generalizable data-driven multicellular model of pancreatic ductal adenocarcinoma</a>.
                        2020.
                    </li>
                    <li>
                        Gibbs DL, Aguilar B, Thorsson V, Ratushny AV, Shmulevich I.
                        <a target="_blank" rel="noreferrer"
                           href="https://storage.googleapis.com/isb-cgc-publications/Patient-Specific%20Cell%20Communication%20Networks%20Associate%20With%20Disease%20Progression%20in%20Cancer.pdf">Patient-Specific
                            Cell Communication Networks Associate With Disease Progression in Cancer.</a>
                        Frontiers in Genetics. 2021;12:1489. doi:10.3389/fgene.2021.667382.
                    </li>
                    <li>
                        Boris Aguilar, Kawther Abdilleh, George Acquaah-Mensah.
                        <a target="_blank" rel="noreferrer"
                           href="https://storage.googleapis.com/isb-cgc-publications/CSHLVirtualMeeting2021-A%20tale%20of%20two%20cohorts%E2%80%95Transcriptomic%20and%20epigenomic%20analysis%20in%20breast%20cancer%20presentations.pdf">A
                            tale of two cohorts: Transcriptomics and epigenomic analysis in breast cancer</a>.
                        2021.
                    </li>
                    <li>
                        Kawther Abdilleh, Boris Aguilar, Ronald C. Taylor, et al.
                        <a target="_blank" rel="noreferrer"
                           href="https://storage.googleapis.com/isb-cgc-publications/Poster_ISMB2020_Acquaah-Mensah.pdf">Multi-omics
                            data analysis in the cloud: inference of differential breast cancer-related network hubs
                            between TCGA patient cohorts</a>.
                        2021. doi: 10.7490/f1000research.1118296.1.
                    </li>
                    <li>
                        Plaugher D, Aguilar B, Murrugarra D.
                        <a target="_blank" rel="noreferrer"
                           href="https://storage.googleapis.com/isb-cgc-publications/Uncovering%20potential%20interventions%20for%20pancreatic%20cancer%20patients%20via%20mathematical%20modeling.pdf">Uncovering
                            potential interventions for pancreatic cancer patients via mathematical modeling</a>. 2022.
                        doi: 10.1101/2022.01.11.475711.
                    </li>
                    <li>
                        de Andrade KC, Lee EE, Tookmanian EM, et al.
                        <a target="_blank" rel="noreferrer" href="https://www.nature.com/articles/s41418-022-00976-3">The
                            TP53 Database: transition from the International Agency for Research on Cancer to the US
                            National Cancer Institute</a>. 2022.
                        doi: 10.1038/s41418-022-00976-3.
                    </li>
                    <li>
                        Tercan B, Qin G, Kim TK, et al.
                        <a target="_blank" rel="noreferrer" href="https://f1000research.com/articles/11-493">SL-Cloud: A
                            Cloud-based resource to support synthetic lethal interaction discovery</a>. 2022.
                        doi:10.12688/f1000research.110903.2.
                    </li>
                    <li>
                        Wang J, Zheng J, Lee E, et al.
                        <a target="_blank" rel="noreferrer"
                           href="https://onlinelibrary.wiley.com/doi/abs/10.1002/gcc.23128">A cloud-based resource for
                            genome coordinate-based exploration and large-scale analysis of chromosome aberrations and
                            gene fusions in cancer</a>. 2023.
                        doi:10.1002/gcc.23128.
                    </li>
                    <li>
                        Torcivia J, Abdilleh K, Seidl F, et al.
                        <a target="_blank" rel="noreferrer" href="https://www.mdpi.com/2673-7523/2/2/9">Whole Genome
                            Variant Dataset for Enriching Studies across 18 Different Cancers</a>. 2023.
                        doi:10.3390/onco2020009.
                    </li>
                    <li>
                        Pot D, Worman Z, Baumann A, et al.
                        <a target="_blank" rel="noreferrer" href="https://pubmed.ncbi.nlm.nih.gov/38488504/">NCI Cancer Research Data Commons: Cloud-based
                            Analytical Resources.</a> Cancer Research.
                        2024. doi:10.1158/0008-5472.CAN-23-2657
                    </li>
                    <li>
                        Seidl F, Hagen L, Wilson J, et al.
                        <a target="_blank" rel="noreferrer" href="https://aacrjournals.org/cancerres/article/84/6_Supplement/3547/740713/Abstract-3547-The-ISB-Cancer-Gateway-in-the-Cloud">The ISB Cancer Gateway in the Cloud (ISB-CGC):
                            Access, explore and analyze large-scale cancer data through the Google Cloud.</a> Cancer
                        Research.
                        2024. doi:10.1158/1538-7445.AM2024-3547
                    </li>
                    <li>
                        Kawther Abdilleh, Boris Aguilar, George Acquaah-Mensah
                        <a target="_blank" rel="noreferrer"
                           href="https://www.sciencedirect.com/science/article/abs/pii/S1526820924003215">
                            Clinical and Multiomic Features Differentiate Young Black and White Breast Cancer Cohorts
                            Derived by Machine Learning Approaches
                        </a>. Clinical Breast Cancer, 2024. doi:10.1016/j.clbc.2024.11.015
                    </li>
                    <li>
                        Gibbs DL, Cioffi G, Aguilar B, et al.
                        <a target="_blank" rel="noreferrer"
                           href="https://www.mdpi.com/2072-6694/17/3/445">Robust
                            Cluster Prediction Across Data Types Validates Association of Sex and Therapy Response in
                            GBM.</a> Cancers, 2025. doi:10.3390/cancers17030445
                    </li>
                </ol>
                <h2>ISB-CGC Blogs</h2>
                <ol>
                    <li>
                        Bleich D, Wilson J. <a target="_blank" rel="noreferrer"
                                               href="https://datascience.cancer.gov/news-events/news/notebook-demonstrates-machine-learning-google-bigquery-using-mitelman-database">New
                        Notebook Demonstrates Machine Learning in Google BigQuery Using Updated Mitelman Database.</a>
                        2024.
                    </li>
                    <li>
                        Thomson R.
                        <a target="_blank" rel="noreferrer"
                           href="https://medium.com/@jrossthomson/how-to-run-statistics-inside-bigquery-95c0c6864f23">How
                            to run statistics inside BigQuery</a>. 2023.
                    </li>
                    <li>
                        Bleich D. <a target="_blank" rel="noreferrer"
                                     href="https://datascience.cancer.gov/news-events/blog/how-mitelman-database-can-help-you-explore-genomic-abnormalities">How
                        the Mitelman Database Can Help You Explore Genomic Abnormalities.</a> 2023.
                    </li>
                    <li>
                        Bleich D. <a target="_blank" rel="noreferrer"
                                     href="https://datascience.cancer.gov/news-events/blog/isb-cgc-cloud-resource-providing-researchers-shortcuts-data-analysis">ISB-CGC
                        Cloud Resource: Providing Researchers with Shortcuts to Data Analysis.</a> 2022.
                    </li>
                    <li>
                        Bleich D, Kuan A, Pot D, Ray M, Subramanian SL, Van der Auwera G. <a target="_blank"
                                                                                             rel="noreferrer"
                                                                                             href="https://datascience.cancer.gov/news-events/blog/ncis-cloud-resources-help-tame-todays-data-windfall">NCI’s
                        Cloud Resources Help Tame Today’s Data Windfall.</a> 2021.
                    </li>

                </ol>
                <h2>Citations</h2>
                <ol>
                    <li>Abad-Carratalà G, Blanco-Silvestre M, Sánchez-Llopis A, Monsonís-Usó R, Martínez-Cadenas C,
                        Martínez-Meneu P, et al. Two Clinical Cases of Li-Fraumeni Syndrome and Prostate Cancer: Genetic
                        Counseling and Clinical-Surgical Management. Clinical Genitourinary Cancer 2022;20:581–5. <a
                                href="https://doi.org/10.1016/j.clgc.2022.06.002" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.clgc.2022.06.002</a>.
                    </li>
                    </li>
                    <li>Abdulrahman FA, Benford KA, Lin GT, Maroun AJ, Sammons C, Shirzad DN, et al. zDHHC-Mediated
                        S-Palmitoylation in Skin Health and Its Targeting as a Treatment Perspective. IJMS 2025;26:1673.
                        <a href="https://doi.org/10.3390/ijms26041673" rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/ijms26041673</a>.
                    </li>
                    <li>Abrams Zachary B, Coombes CE, Li S, Coombes KR. Mercator: a pipeline for multi-method,
                        unsupervised visualization and distance generation. Bioinformatics 2021. <a
                                href="https://doi.org/10.1093/bioinformatics/btab037" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1093/bioinformatics/btab037</a>.
                    </li>
                    <li>Abrams Zachary B., Tally DG, Zhang L, Coombes CE, Payne PRO, Abruzzo LV, et al. Pattern
                        recognition in lymphoid malignancies using CytoGPS and Mercator. BMC Bioinformatics 2021;22. <a
                                href="https://doi.org/10.1186/s12859-021-03992-1" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1186/s12859-021-03992-1</a>.
                    </li>
                    <li>Adam R, Catchpoole DR, Simoff SS, Kennedy PJ, Nguyen QV. Novel Hybrid Edge-Cloud Framework for
                        Efficient and Sustainable Omics Data Management. Innovations in Digital Health, Diagnostics, and
                        Biomarkers 2024;4:81–8. <a href="https://doi.org/10.36401/IDDB-24-5" rel="noreferrer nofollow"
                                                   target="_blank">https://doi.org/10.36401/IDDB-24-5</a>.
                    </li>
                    <li>Afshari MK. Transcriptomic and functional studies of fusion oncogene-driven salivary gland
                        tumors. 2020.
                    </li>
                    <li>Aguilar B, Abdilleh K, Acquaah-Mensah G. Multi-omics inference of differential breast
                        cancer-related transcriptional regulatory network gene hubs between young Black and White
                        patients. Cancer Genetics 2022. <a href="https://doi.org/10.1016/j.cancergen.2022.11.001"
                                                           rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.cancergen.2022.11.001</a>.
                    </li>
                    <li>Ahmad S, Khan FN, Ramlal A, Begum S, Qazi S, Raza K. Chapter 11 - Nanoinformatics and
                        nanomodeling: Recent developments in computational nanodrug design and delivery systems. In:
                        Ahmad N, Packirisamy G, editors. Emerging Nanotechnologies for Medical Applications, Elsevier;
                        2023, p. 297–332. <a href="https://doi.org/10.1016/B978-0-323-91182-5.00001-2"
                                             rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/B978-0-323-91182-5.00001-2</a>.
                    </li>
                    <li>Ahmad ST, Li Y, Garcia-Lopez J, Gudenas BL, Hadley J, Paul L, et al. Genetic modeling of
                        ELP1-associated Sonic hedgehog medulloblastoma identifies MDM2 as a selective therapeutic
                        target. Cancer Cell 2025:S1535610825001734. <a
                                href="https://doi.org/10.1016/j.ccell.2025.04.014" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.ccell.2025.04.014</a>.
                    </li>
                    <li>Ahmadi SE, Rahimian E, Rahimi S, Zarandi B, Bahraini M, Soleymani M, et al. From regulation to
                        deregulation of p53 in hematologic malignancies: implications for diagnosis, prognosis and
                        therapy. Biomark Res 2024;12:137. <a href="https://doi.org/10.1186/s40364-024-00676-9"
                                                             rel="noreferrer nofollow" target="_blank">https://doi.org/10.1186/s40364-024-00676-9</a>.
                    </li>
                    <li>Ajayi TA, Okeke CC, Ogunbajo OO, Ayams JN, Oyewole OA, Adetunji CO, et al. Bioinformatics,
                        instrumentation, and control for modeling, synthesis, and characterization of chitosan based
                        nanoparticles. Chitosan-Based Nanoparticles for Biomedical Applications, Elsevier; 2025, p.
                        17–31. <a href="https://doi.org/10.1016/B978-0-443-13997-0.00002-3" rel="noreferrer nofollow"
                                  target="_blank">https://doi.org/10.1016/B978-0-443-13997-0.00002-3</a>.
                    </li>
                    <li>Akhila. Acute myeloid leukemia patients with variant or unusual translocations involving
                        chromosomes 8 and 21 – A comprehensive cytogenetic profiling of three cases with review of
                        literature 2022.
                        https://www.cancerjournal.net/article.asp?issn=0973-1482;year=2022;volume=18;issue=3;spage=697;epage=703;aulast=Akhila
                        (accessed August 15, 2022).
                    </li>
                    <li>Akkari Y, Baughn LB, Kim A, Karaca E, Raca G, Shao L, et al. Section E6.1–6.6 of the American
                        College of Medical Genetics and Genomics (ACMG) Technical Laboratory Standards: Cytogenomic
                        studies of acquired chromosomal abnormalities in neoplastic blood, bone marrow, and lymph nodes.
                        Genetics in Medicine 2024:101054. <a href="https://doi.org/10.1016/j.gim.2023.101054"
                                                             rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.gim.2023.101054</a>.
                    </li>
                    <li>Akyüz N, Janjetovic S, Ghandili S, Bokemeyer C, Dierlamm J. EBV and 1q Gains Affect Gene and
                        miRNA Expression in Burkitt Lymphoma. Viruses 2023;15:1808. <a
                                href="https://doi.org/10.3390/v15091808" rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/v15091808</a>.
                    </li>
                    <li>Albashir MH, Aljohani SAS, Aloufi K, Hamad AM, Mustafa EH, Yousif SA, et al. AI-DRIVEN
                        PREDICTION OF NEUROPSYCHOLOGICAL RESPONSES TO NANOPARTICLE EXPOSURE VIA INTEGRATION OF
                        NANOPARTICLE PROPERTIES, MICROBIOME PROFILES, AND GENOMIC DATA. Open Access 2025;32.
                    </li>
                    <li>Allers S, O’Connell KA, Carlson T, Belardo D, King BL. Reusable tutorials for using cloud-based
                        computing environments for the analysis of bacterial gene expression data from bulk RNA
                        sequencing. Briefings in Bioinformatics 2024;25:bbae301. <a
                                href="https://doi.org/10.1093/bib/bbae301" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1093/bib/bbae301</a>.
                    </li>
                    <li>Amatruda JF, Houart C, Kawakami K, Poss KD. Zebrafish: Methods and Protocols. Springer Nature;
                        2023.
                    </li>
                    <li>Ammar A. Enhancing the interoperability and reusability of nanosafety data: A foundation towards
                        deeper insights and intelligent applications. maastricht university, 2025. <a
                                href="https://doi.org/10.26481/dis.20250702aa" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.26481/dis.20250702aa</a>.
                    </li>
                    <li>Ananthaneni A, Benzar T, Hafiz N, Akabane H. De novo monocytic-M5b AML with t(8;16) (p11.2;
                        p13.3) KAT6A/CREBBP fusion and FLT3-TKD mutation complicated by chemotherapy-induced Takotsubo
                        cardiomyopathy. BMJ Case Reports CP 2023;16:e253812. <a
                                href="https://doi.org/10.1136/bcr-2022-253812" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1136/bcr-2022-253812</a>.
                    </li>
                    <li>Andrade J, Cox SM, Volchenboum SL. Large-Scale Data Sharing Initiatives in Genomic Oncology.
                        Advances in Molecular Pathology 2018;1:135–48. <a
                                href="https://doi.org/10.1016/j.yamp.2018.06.009" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.yamp.2018.06.009</a>.
                    </li>
                    <li>Andrew B, Attilio O, Sumit G, Vijay T, Reshad GS, Osvaldo P. A rare case of B cell lymphoblastic
                        leukemia with inv(7)(p15q34) with review of literature. Leukemia Research Reports
                        2021;15:100250. <a href="https://doi.org/10.1016/j.lrr.2021.100250" rel="noreferrer nofollow"
                                           target="_blank">https://doi.org/10.1016/j.lrr.2021.100250</a>.
                    </li>
                    <li>Anitha DC, Gnanatheepa KTC, Hubert G, Yadav R, Naik SB, Singh B. AI BASED HERBAL TREATMENT FOR
                        CANCER CELL. Journal of Pharmaceutical Negative Results 2022:7623–36. <a
                                href="https://doi.org/10.47750/pnr.2022.13.S07.920" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.47750/pnr.2022.13.S07.920</a>.
                    </li>
                    <li>Arbajian E, Hofvander J, Magnusson L, Mertens F. Deep sequencing of myxoinflammatory
                        fibroblastic sarcoma. Genes Chromosomes and Cancer 2020;59:309–17. <a
                                href="https://doi.org/10.1002/gcc.22832" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1002/gcc.22832</a>.
                    </li>
                    <li>Arnon J, Zick A, Maoz M, Salaymeh N, Gugenheim A, Marouani M, et al. Clinical and genetic
                        characteristics of carriers of the TP53 c.541C > T, p.Arg181Cys pathogenic variant causing
                        hereditary cancer in patients of Arab-Muslim descent. Familial Cancer 2024. <a
                                href="https://doi.org/10.1007/s10689-024-00391-2" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1007/s10689-024-00391-2</a>.
                    </li>
                    <li>Avenarius MR, Abrams ZB, Guo L, Blachly JS, Miller CR, Heerema NA, et al. Novel Recurrent
                        Cytogenetic Abnormalities Predict Overall Survival in Tetraploid/Near-Tetraploid Myelodysplastic
                        Syndrome and Acute Myeloid Leukemia. Cancers 2025;17:1277. <a
                                href="https://doi.org/10.3390/cancers17081277" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.3390/cancers17081277</a>.
                    </li>
                    <li>Avgerinou G, Stefanaki K, Liapis K, Kostopoulos IV, Kossiva L, Tzoumaka-Bakoula C, et al. Fish
                        evaluation of additional cytogenetic aberrations and hyperdiploidy in childhood Burkitt
                        lymphoma. Leukemia & Lymphoma 2021;0:1–11. <a
                                href="https://doi.org/10.1080/10428194.2021.1998480" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1080/10428194.2021.1998480</a>.
                    </li>
                    <li>Awada Hussein, Durmaz A, Kewan T, Ullah F, Dima D, Awada Hassan, et al. Context-dependent role
                        of trisomy 6 in myelodysplastic neoplasms and acute myeloid leukemia: a multi-omics analysis.
                        Leukemia 2024;38:1411–4. <a href="https://doi.org/10.1038/s41375-024-02268-w"
                                                    rel="noreferrer nofollow" target="_blank">https://doi.org/10.1038/s41375-024-02268-w</a>.
                    </li>
                    <li>Bahrami Hezaveh E, Yan J, Zhao D, Wangulu C, Lo W, Wei C, et al. Genetic Landscape and Risk
                        Stratification of AML With Hyperdiploid Karyotype. European J of Haematology 2025:ejh.14434. <a
                                href="https://doi.org/10.1111/ejh.14434" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1111/ejh.14434</a>.
                    </li>
                    <li>Balla B, Tripon F, Candea M, Banescu C. Copy Number Variations and Gene Mutations Identified by
                        Multiplex Ligation-Dependent Probe Amplification in Romanian Chronic Lymphocytic Leukemia
                        Patients. Journal of Personalized Medicine 2023;13:1239. <a
                                href="https://doi.org/10.3390/jpm13081239" rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/jpm13081239</a>.
                    </li>
                    <li>Balourdas D-I, Markl AM, Krämer A, Settanni G, Joerger AC. Structural basis of p53 inactivation
                        by cavity-creating cancer mutations and its implications for the development of mutant p53
                        reactivators. Cell Death Dis 2024;15:408. <a href="https://doi.org/10.1038/s41419-024-06739-x"
                                                                     rel="noreferrer nofollow" target="_blank">https://doi.org/10.1038/s41419-024-06739-x</a>.
                    </li>
                    <li>Balzano E. Common Fragile Sites: a new tool to study chromosome instability diseases 2023.
                        https://phd.uniroma1.it/web/ELISA-BALZANO_nT1467247_EN.aspx (accessed February 7, 2023).
                    </li>
                    <li>Bang B, Eisfeldt J, Barbany G, Harila-Saari A, Heyman M, Zachariadis V, et al. A somatic UBA2
                        variant preceded ETV6-RUNX1 in the concordant BCP-ALL of monozygotic twins. Blood Advances
                        2022:bloodadvances.2021005703. <a href="https://doi.org/10.1182/bloodadvances.2021005703"
                                                          rel="noreferrer nofollow" target="_blank">https://doi.org/10.1182/bloodadvances.2021005703</a>.
                    </li>
                    <li>Barahuie F, Dorniani D, Saifullah B, Arulselvan P, Hussein MZ, Jaganathan R, et al. Tannic acid
                        chitosan iron oxide nanocomposite for cervical cancer treatment. Inorganic Chemistry
                        Communications 2024;162:112160. <a href="https://doi.org/10.1016/j.inoche.2024.112160"
                                                           rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.inoche.2024.112160</a>.
                    </li>
                    <li>Basu K, Chelagamsetty VS, Ruiz-Avila VA, Li T. Machine Learning Prediction of Protein Adsorption
                        on Drug-delivering Nanoparticles: A Literature Survey and Need for Future Development. Pharm Res
                        2025. <a href="https://doi.org/10.1007/s11095-025-03981-6" rel="noreferrer nofollow"
                                 target="_blank">https://doi.org/10.1007/s11095-025-03981-6</a>.
                    </li>
                    <li>Bazylevych N, Shtanagey D, Chuprun N, Bozhko S, Zakopailo S, Polishchuk V, et al. Prognosis and
                        Functional Analyzes of Missense Mutations in Exon 4 of the TP53 Gene in Colorectal Cancer in the
                        Senegalese Population. J Genet Genet Eng 2025;7:13–29. <a
                                href="https://doi.org/10.22259/2637-5370.0701003" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.22259/2637-5370.0701003</a>.
                    </li>
                    <li>Behrens YL, Pietzsch S, Antić Ž, Zhang Y, Bergmann AK. The landscape of cytogenetic and
                        molecular genetic methods in diagnostics for hematologic neoplasia. Best Practice & Research
                        Clinical Haematology 2024;37:101539. <a href="https://doi.org/10.1016/j.beha.2024.101539"
                                                                rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.beha.2024.101539</a>.
                    </li>
                    <li>Behrens YL, Schienke A, Davenport C, Lentes J, Tauscher M, Steinemann D, et al. BCR-ABL1
                        positive AML or CML in blast crisis? A pediatric case report with inv(3) and t(9;22) in the
                        initial clone. Cancer Genetics 2021;254–255:70–4. <a
                                href="https://doi.org/10.1016/j.cancergen.2021.02.007" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.cancergen.2021.02.007</a>.
                    </li>
                    <li>Bernasconi A, Canakoglu A, Masseroli M, Ceri S. The road towards data integration in human
                        genomics: players, steps and interactions. Briefings in Bioinformatics 2021;22:30–44. <a
                                href="https://doi.org/10.1093/bib/bbaa080" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1093/bib/bbaa080</a>.
                    </li>
                    <li>Bernués M, González T, Corchete LA, Santos S, Durán MA, López-Andrade B, et al.
                        t(10;12)(q24;q15): A new cytogenetic marker in hematological malignancies. Cancer Genetics
                        2022;264–265:60–5. <a href="https://doi.org/10.1016/j.cancergen.2022.03.004"
                                              rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.cancergen.2022.03.004</a>.
                    </li>
                    <li>Biloglav A, Olsson‐Arvidsson L, Theander J, Behrendtz M, Castor A, Johansson B. SFPQ‐ABL1
                        ‐positive B‐cell precursor acute lymphoblastic leukemias. Genes, Chromosomes and Cancer
                        2020:gcc.22852. <a href="https://doi.org/10.1002/gcc.22852" rel="noreferrer nofollow"
                                           target="_blank">https://doi.org/10.1002/gcc.22852</a>.
                    </li>
                    <li>Blakemore C, Damodharan S, Puccetti D. Inv(3) Acute Myeloid Leukemia in a Young Adult and Review
                        of the Literature. Case Reports in Oncological Medicine 2023;2023:e6628492. <a
                                href="https://doi.org/10.1155/2023/6628492" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1155/2023/6628492</a>.
                    </li>
                    <li>Bleich D. How the Mitelman Database Can Help You Explore Genomic Abnormalities 2023.
                        https://datascience.cancer.gov/news-events/blog/how-mitelman-database-can-help-you-explore-genomic-abnormalities.
                    </li>
                    <li>Bleich D. ISB-CGC Cloud Resource: Providing Researchers with Shortcuts to Data Analysis 2022.
                        https://datascience.cancer.gov/news-events/blog/isb-cgc-cloud-resource-providing-researchers-shortcuts-data-analysis.
                    </li>
                    <li>Bleich D, Kuan A, Pot D, Ray M, Subramanian SL, Van der Auwera G. NCI’s Cloud Resources Help
                        Tame Today’s Data Windfall 2021.
                        https://datascience.cancer.gov/news-events/blog/ncis-cloud-resources-help-tame-todays-data-windfall.
                    </li>
                    <li>Bleich D, Wilson J. New Notebook Demonstrates Machine Learning in Google BigQuery Using Updated
                        Mitelman Database 2024.
                        https://datascience.cancer.gov/news-events/news/notebook-demonstrates-machine-learning-google-bigquery-using-mitelman-database.
                    </li>
                    <li>Bomben R, Zucchetto A, Pozzo F, Tissino E, Bittolo T, Olivieri J, et al. TP53 Mutations and
                        Clinical Outcome in Chronic Lymphocytic Leukemia: Is a Threshold Still Needed? HemaSphere
                        2023;7:e855. <a href="https://doi.org/10.1097/HS9.0000000000000855" rel="noreferrer nofollow"
                                        target="_blank">https://doi.org/10.1097/HS9.0000000000000855</a>.
                    </li>
                    <li>Brockman KJ, Thompson MB, Mirabello L, Savage SA, Malayeri A, Hatton JN, et al. Characterization
                        of sarcoma topography in Li-Fraumeni syndrome. Front Oncol 2024;14:1415636. <a
                                href="https://doi.org/10.3389/fonc.2024.1415636" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.3389/fonc.2024.1415636</a>.
                    </li>
                    <li>Bruford EA, Antonescu CR, Carroll AJ, Chinnaiyan A, Cree IA, Cross NCP, et al. HUGO Gene
                        Nomenclature Committee (HGNC) recommendations for the designation of gene fusions. Leukemia
                        2021;35:3040–3. <a href="https://doi.org/10.1038/s41375-021-01436-6" rel="noreferrer nofollow"
                                           target="_blank">https://doi.org/10.1038/s41375-021-01436-6</a>.
                    </li>
                    <li>Brukman-Jimenez SA, Bobadilla-Morales L, Corona-Rivera JR, Chávez-Panduro PA, Ortega-de-la-Torre
                        C, Santana-Bejarano UF, et al. Sole trisomy 6 an uncommon finding in pediatric acute myeloid
                        leukemia, probably associated to bad prognosis. Molecular Cytogenetics 2020;13:41. <a
                                href="https://doi.org/10.1186/s13039-020-00509-y" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1186/s13039-020-00509-y</a>.
                    </li>
                    <li>Brunetti M, Panagopoulos I, Vitelli V, Andersen K, Hveem TS, Davidson B, et al. Endometrial
                        Carcinoma: Molecular Cytogenetics and Transcriptomic Profile. Cancers 2022;14:3536. <a
                                href="https://doi.org/10.3390/cancers14143536" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.3390/cancers14143536</a>.
                    </li>
                    <li>Brunetti, M.; Zeller, B.;Tierens, A.;Heim, S.;Micci, F.; Panagopoulos I. TYRO3 Truncation
                        Resulting From a t(10;15)(p11;q15) Chromosomal Translocation in Pediatric Acute Myeloid
                        Leukemia. Anticancer Research 2020;40:6115–21.
                    </li>
                    <li>Bueschbell B, Caniceiro AB, Suzano PMS, Machuqueiro M, Rosário-Ferreira N, Moreira IS. Network
                        biology and artificial intelligence drive the understanding of the multidrug resistance
                        phenotype in cancer. Drug Resistance Updates 2022;60:100811. <a
                                href="https://doi.org/10.1016/j.drup.2022.100811" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.drup.2022.100811</a>.
                    </li>
                    <li>C Varelas, G P, Gavriilaki E, M G, Touloumenidou T, C Vadikoliou, et al. Long Term Survival of
                        Chronic Myeloid Leukemia Patients with Chromosomal Aberrations in Philadelphia Negative
                        Metaphases During Treatment with Tyrosine Kinase Inhibitors. Austin Journal of Clinical Medicine
                        2022;8. <a href="https://doi.org/10.26420/austinjclinmed.2022.1043" rel="noreferrer nofollow"
                                   target="_blank">https://doi.org/10.26420/austinjclinmed.2022.1043</a>.
                    </li>
                    <li>Canoy RJ, Shmakova A, Karpukhina A, Shepelev M, Germini D, Vassetzky Y. Factors That Affect the
                        Formation of Chromosomal Translocations in Cells. Cancers 2022;14:5110. <a
                                href="https://doi.org/10.3390/cancers14205110" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.3390/cancers14205110</a>.
                    </li>
                    <li>Cao S, Zhou DC, Oh C, Jayasinghe RG, Zhao Y, Yoon CJ, et al. Discovery of driver non-coding
                        splice-site-creating mutations in cancer. Nature Communications 2020;11:5573. <a
                                href="https://doi.org/10.1038/s41467-020-19307-6" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41467-020-19307-6</a>.
                    </li>
                    <li>Cervera Taboada A. Transcriptomics analysis and its applications in cancer. 2020.</li>
                    <li>Chang LC, Vural S, Sonkin D. Detection of homozygous deletions in tumor-suppressor genes ranging
                        from dozen to hundreds nucleotides in cancer models. Human Mutation 2017;38:1449–53. <a
                                href="https://doi.org/10.1002/humu.23308" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1002/humu.23308</a>.
                    </li>
                    <li>Charlab R, Racz R. The expanding universe of NUTM1 fusions in pediatric cancer. Clinical and
                        Translational Science 2023;n/a. <a href="https://doi.org/10.1111/cts.13535"
                                                           rel="noreferrer nofollow" target="_blank">https://doi.org/10.1111/cts.13535</a>.
                    </li>
                    <li>Chebly A, Djambas Khayat C, Yammine T, Korban R, Semaan W, Bou Zeid J, et al. Pediatric M5 acute
                        myeloid leukemia with MLL-SEPT6 fusion and a favorable outcome. Leukemia Research Reports
                        2021;16:100277. <a href="https://doi.org/10.1016/j.lrr.2021.100277" rel="noreferrer nofollow"
                                           target="_blank">https://doi.org/10.1016/j.lrr.2021.100277</a>.
                    </li>
                    <li>Chen Z, Kim E, Davidsen T, Barnholtz-Sloan JS. Usage of the National Cancer Institute Cancer
                        Research Data Commons by Researchers: A Scoping Review of the Literature. JCO Clin Cancer Inform
                        2024:e2400116. <a href="https://doi.org/10.1200/CCI.24.00116" rel="noreferrer nofollow"
                                          target="_blank">https://doi.org/10.1200/CCI.24.00116</a>.
                    </li>
                    <li>Cheng C-K, Yung Y-L, Chan H-Y, Leung K-T, Chan KYY, Leung AWK, et al. Deep genomic
                        characterization highlights complexities and prognostic markers of pediatric acute myeloid
                        leukemia. Commun Biol 2023;6:1–16. <a href="https://doi.org/10.1038/s42003-023-04732-2"
                                                              rel="noreferrer nofollow" target="_blank">https://doi.org/10.1038/s42003-023-04732-2</a>.
                    </li>
                    <li>Chou W-C, Canchola A, Zhang F, Lin Z. Machine Learning and Artificial Intelligence in
                        Nanomedicine. WIREs Nanomedicine and Nanobiotechnology 2025;17:e70027. <a
                                href="https://doi.org/10.1002/wnan.70027" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1002/wnan.70027</a>.
                    </li>
                    <li>Chung YH, Ortega-Rivera OA, Volckaert BA, Jung E, Zhao Z, Steinmetz NF. Viral nanoparticle
                        vaccines against S100A9 reduce lung tumor seeding and metastasis. Proc Natl Acad Sci USA
                        2023;120:e2221859120. <a href="https://doi.org/10.1073/pnas.2221859120"
                                                 rel="noreferrer nofollow" target="_blank">https://doi.org/10.1073/pnas.2221859120</a>.
                    </li>
                    <li>Cirakoglu A, Kuru RD, Yilmaz S, Deviren A, Ongoren S, Yalniz FF, et al. Cytogenetic profile of
                        adult AML patients in Turkey: a single center study with comprehensive comparison with
                        literature. African Health Sciences 2022;22:183–91.
                    </li>
                    <li>Clogston JD, Crist RM, Dobrovolskaia MA, Stern ST, editors. Characterization of Nanoparticles
                        Intended for Drug Delivery. vol. 2789. New York, NY: Springer US; 2024. <a
                                href="https://doi.org/10.1007/978-1-0716-3786-9" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1007/978-1-0716-3786-9</a>.
                    </li>
                    <li>Closa A, Reixachs-Sole M, Fuentes-Fayos AC, Hayer KE, Melero JL, Adriaanse FRS, et al. A
                        convergent malignant phenotype in B-cell acute lymphoblastic leukemia involving the splicing
                        factor SRRM 2022.
                    </li>
                    <li>Colombo C, Pogliaghi G, Tosi D, Muzza M, Bulfamante G, Persani L, et al. Thyroid cancer
                        harboring PTEN and TP53 mutations: A peculiar molecular and clinical case report. Front Oncol
                        2022;12:949098. <a href="https://doi.org/10.3389/fonc.2022.949098" rel="noreferrer nofollow"
                                           target="_blank">https://doi.org/10.3389/fonc.2022.949098</a>.
                    </li>
                    <li>Costa D, Granada I, Espinet B, Collado R, Ruiz-Xivillé N, Puiggros A, et al. Balanced and
                        unbalanced translocations in a multicentric series of 2,843 patients with chronic lymphocytic
                        leukemia. Genes, Chromosomes and Cancer 2021;n/a. <a href="https://doi.org/10.1002/gcc.22994"
                                                                             rel="noreferrer nofollow" target="_blank">https://doi.org/10.1002/gcc.22994</a>.
                    </li>
                    <li>Curylova L. The p53 family proteins in the regulation of sternness in pediatric sarcomas.
                        Masaryk University, 2025.
                    </li>
                    <li>Daifuku R, Zhang Y, Wang J, Gu Q. iMN041 is an immunotherapeutic and an effective treatment in
                        mouse xenograft models of pancreatic cancer, renal cancer and triple negative breast cancer.
                        Transl Med Commun 2024;9:2. <a href="https://doi.org/10.1186/s41231-024-00161-3"
                                                       rel="noreferrer nofollow" target="_blank">https://doi.org/10.1186/s41231-024-00161-3</a>.
                    </li>
                    <li>Danishevich A, Fedorova D, Bodunova N, Makarova M, Byakhova M, Semenova A, et al. Assessing
                        germline TP53 mutations in cancer patients: insights into Li-Fraumeni syndrome and genetic
                        testing guidelines. Hered Cancer Clin Pract 2025;23:5. <a
                                href="https://doi.org/10.1186/s13053-025-00307-w" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1186/s13053-025-00307-w</a>.
                    </li>
                    <li>Datta S, Patel M, Kashyap S, Patel D, Singh U. Chimeric chromosome landscapes of human somatic
                        cell cultures show dependence on stress and regulation of genomic repeats by CGGBP1. Oncotarget
                        2022;13:136–55. <a href="https://doi.org/10.18632/oncotarget.28174" rel="noreferrer nofollow"
                                           target="_blank">https://doi.org/10.18632/oncotarget.28174</a>.
                    </li>
                    <li>Dawoud AAZ. The Causes and Consequences of Clonal Haematopoiesis 2023.</li>
                    <li>De Luca G, Cerruti G, Lastraioli S, Conte R, Ibatici A, Di Felice N, et al. The spectrum of
                        subclonal TP53 mutations in chronic lymphocytic leukemia: A next generation sequencing
                        retrospective study. Hematological Oncology 2022;n/a. <a href="https://doi.org/10.1002/hon.3063"
                                                                                 rel="noreferrer nofollow"
                                                                                 target="_blank">https://doi.org/10.1002/hon.3063</a>.
                    </li>
                    <li>Decarvalho AC, Kim H, Poisson LM, Winn ME, Mueller C, Cherba D, et al. Discordant inheritance of
                        chromosomal and extrachromosomal DNA elements contributes to dynamic disease evolution in
                        glioblastoma. Nature Genetics 2018;50:708–17. <a
                                href="https://doi.org/10.1038/s41588-018-0105-0" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41588-018-0105-0</a>.
                    </li>
                    <li>Decock A, Creytens D, Lefever S, Van der Meulen J, Anckaert J, De Ganck A, et al. mRNA Capture
                        Sequencing and RT-qPCR for the Detection of Pathognomonic, Novel, and Secondary Fusion
                        Transcripts in FFPE Tissue: A Sarcoma Showcase. International Journal of Molecular Sciences
                        2022;23:11007. <a href="https://doi.org/10.3390/ijms231911007" rel="noreferrer nofollow"
                                          target="_blank">https://doi.org/10.3390/ijms231911007</a>.
                    </li>
                    <li>Deng H, Hung L-H, Schooley R, Perez D, Arumilli N, Yeung KY, et al. Profiling Resource
                        Utilization of Bioinformatics Workflows. arXiv:200511491 [Cs] 2020.
                    </li>
                    <li>Denu RA, Yang RK, Lazar AJ, Patel SS, Lewis VO, Roszik J, et al. Clinico-genomic profiling of
                        conventional and dedifferentiated chondrosarcomas reveals TP53 mutation to be associated with
                        worse outcomes. Clinical Cancer Research 2023. <a
                                href="https://doi.org/10.1158/1078-0432.CCR-23-1703" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1158/1078-0432.CCR-23-1703</a>.
                    </li>
                    <li>Difilippo V. Decoding Genetic Enigmas in Sarcoma. Doctoral Thesis (compilation). Lund
                        University, Faculty of Medicine, 2024.
                    </li>
                    <li>Difilippo V, Saba KH, Styring E, Magnusson L, Nilsson J, Nathrath M, et al. Osteosarcomas with
                        few chromosomal alterations and/or affecting adults are genetically heterogeneous. Laboratory
                        Investigation 2023;0. <a href="https://doi.org/10.1016/j.labinv.2023.100283"
                                                 rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.labinv.2023.100283</a>.
                    </li>
                    <li>Diot-Dejonghe T, Leporq B, Bouhamama A, Ratiney H, Pilleul F, Beuf O, et al. Development of a
                        Secure Web-Based Medical Imaging Analysis Platform: The AWESOMME Project. J Digit Imaging Inform
                        Med 2024. <a href="https://doi.org/10.1007/s10278-024-01110-0" rel="noreferrer nofollow"
                                     target="_blank">https://doi.org/10.1007/s10278-024-01110-0</a>.
                    </li>
                    <li>Donaldson B. Reciprocal Chromosome Translocations in the Domestic Pig, the Prevalence, Genetic
                        and Genomic Factors Associated with Breakpoint Formation. PhD Thesis. University of Guelph,
                        2020.
                    </li>
                    <li>Đorđević A. NANOINFORMATICS, COMPUTER APPLICATIONS FOR NANOMEDICINE. Social Informatics Journal
                        2023;2:27–32. <a href="https://doi.org/10.58898/sij.v2i1.27-32" rel="noreferrer nofollow"
                                         target="_blank">https://doi.org/10.58898/sij.v2i1.27-32</a>.
                    </li>
                    <li>Dorwal P, White C, Goh AF, Kumar A, McEniery J, Walker R, et al. Ependymoma-like tumor with
                        mesenchymal differentiation (ELTMD) with ZFTA:NCOA1 fusion: A diagnostic challenge.
                        Neuropathology 2023;n/a. <a href="https://doi.org/10.1111/neup.12952" rel="noreferrer nofollow"
                                                    target="_blank">https://doi.org/10.1111/neup.12952</a>.
                    </li>
                    <li>DSouza D, Bik L, Giwa O, Cohen S, Barazany HL, Siegal T, et al. ChiTaRS 8.0: the comprehensive
                        database of chimeric transcripts and RNA-seq data with applications in liquid biopsy. Nucleic
                        Acids Research 2024:gkae1126. <a href="https://doi.org/10.1093/nar/gkae1126"
                                                         rel="noreferrer nofollow" target="_blank">https://doi.org/10.1093/nar/gkae1126</a>.
                    </li>
                    <li>Du M, Cai Q, Sun J, Zhang Mingxing, Zhang S, Liu X, et al. Aneuploid serves as a prognostic
                        marker and favors immunosuppressive microenvironment in ovarian cancer. Journal of Ovarian
                        Research 2024;17:30. <a href="https://doi.org/10.1186/s13048-024-01356-w"
                                                rel="noreferrer nofollow" target="_blank">https://doi.org/10.1186/s13048-024-01356-w</a>.
                    </li>
                    <li>Duan J, Liu F, Chang L, Che G, Yang Q, Teng J, et al. A primary pediatric acute myelomonocytic
                        leukemia with t(3;21)(q26;q22): A case report. Medicine 2023;102:e35721. <a
                                href="https://doi.org/10.1097/MD.0000000000035721" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1097/MD.0000000000035721</a>.
                    </li>
                    <li>Duncan JL, Bloomfield M, Swami N, Cimini D, Davalos RV. High-Frequency Dielectrophoresis Reveals
                        That Distinct Bio-Electric Signatures of Colorectal Cancer Cells Depend on Ploidy and Nuclear
                        Volume. Micromachines 2023;14:1723. <a href="https://doi.org/10.3390/mi14091723"
                                                               rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/mi14091723</a>.
                    </li>
                    <li>Dunsche L, Ivanisenko N, Riemann S, Schindler S, Beissert S, Angeli C, et al. A cytosolic
                        mutp53(E285K) variant confers chemoresistance of malignant melanoma. Cell Death Dis
                        2023;14:1–12. <a href="https://doi.org/10.1038/s41419-023-06360-4" rel="noreferrer nofollow"
                                         target="_blank">https://doi.org/10.1038/s41419-023-06360-4</a>.
                    </li>
                    <li>Eddy JA, Thorsson V, Lamb AE, Gibbs DL, Heimann C, Yu JX, et al. CRI iAtlas: an interactive
                        portal for immuno-oncology research [version 1; peer review: awaiting peer review].
                        F1000Research 2020;9. <a href="https://doi.org/10.12688/f1000research.25141.1"
                                                 rel="noreferrer nofollow" target="_blank">https://doi.org/10.12688/f1000research.25141.1</a>.
                    </li>
                    <li>Eklund EA, Svensson J, Näslund LS, Yhr M, Sayin SI, Wiel C, et al. Comprehensive genetic variant
                        analysis reveals combination of KRAS and LRP1B as a predictive biomarker of response to
                        immunotherapy in patients with non-small cell lung cancer. J Exp Clin Cancer Res 2025;44:75. <a
                                href="https://doi.org/10.1186/s13046-025-03342-6" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1186/s13046-025-03342-6</a>.
                    </li>
                    <li>Ellrott K, Bailey MH, Saksena G, Covington KR, Kandoth C, Stewart C, et al. Scalable Open
                        Science Approach for Mutation Calling of Tumor Exomes Using Multiple Genomic Pipelines. Cell
                        Systems 2018;6:271-281.e7. <a href="https://doi.org/10.1016/j.cels.2018.03.002"
                                                      rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.cels.2018.03.002</a>.
                    </li>
                    <li>Ellrott K, Buchanan A, Creason A, Mason M, Schaffter T, Hoff B, et al. Reproducible biomedical
                        benchmarking in the cloud: Lessons from crowd-sourced data challenges. Genome Biology 2019;20.
                        <a href="https://doi.org/10.1186/s13059-019-1794-0" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1186/s13059-019-1794-0</a>.
                    </li>
                    <li>Elmarakeby HA, Hwang J, Arafeh R, Crowdis J, Gang S, Liu D, et al. Biologically informed deep
                        neural network for prostate cancer discovery. Nature 2021:1–5. <a
                                href="https://doi.org/10.1038/s41586-021-03922-4" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41586-021-03922-4</a>.
                    </li>
                    <li>Elsner F, Hoffmann M, Fahrioglu-Yamaci R, Czyz Z, Feliciello G, Mederer T, et al. Disseminated
                        cancer cells detected by immunocytology in lymph nodes of NSCLC patients are highly prognostic
                        and undergo parallel molecular evolution. The Journal of Pathology 2022;n/a. <a
                                href="https://doi.org/10.1002/path.5996" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1002/path.5996</a>.
                    </li>
                    <li>Erfannia L, Alipour J. How does cloud computing improve cancer information management? A
                        systematic review. Informatics in Medicine Unlocked 2022;33:101095. <a
                                href="https://doi.org/10.1016/j.imu.2022.101095" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.imu.2022.101095</a>.
                    </li>
                    <li>Fagnan A, Mercher T. NUP98 and KMT2A: Usually the bride rather than the bridesmaid. vol. 105.
                        Ferrata Storti Foundation; 2020. <a href="https://doi.org/10.3324/haematol.2020.253476"
                                                            rel="noreferrer nofollow" target="_blank">https://doi.org/10.3324/haematol.2020.253476</a>.
                    </li>
                    <li>Fan WZ. Genomics and Epidemiological Analysis of Melanoma Laterality. 2018.</li>
                    <li>Fedorov A, Longabaugh WJR, Pot D, Clunie DA, Pieper S, Aerts HJWL, et al. NCI Imaging Data
                        Commons. Cancer Res 2021. <a href="https://doi.org/10.1158/0008-5472.CAN-21-0950"
                                                     rel="noreferrer nofollow" target="_blank">https://doi.org/10.1158/0008-5472.CAN-21-0950</a>.
                    </li>
                    <li>Feng R, Yin Y, Wei Y, Li Y, Li L, Zhu R, et al. Mutant p53 activates hnRNPA2B1-AGAP1-mediated
                        exosome formation to promote esophageal squamous cell carcinoma progression. Cancer Letters
                        2023;562:216154. <a href="https://doi.org/10.1016/j.canlet.2023.216154"
                                            rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.canlet.2023.216154</a>.
                    </li>
                    <li>Fernandes S, Fragoso S, Filipe B, Santos S, Duarte T, Bento S, et al. Diagnosis, follow up and
                        clinical management of individuals identified with a TP53 pathogenic variant. Medical Research
                        Archives 2022;10. <a href="https://doi.org/10.18103/mra.v10i7.2846" rel="noreferrer nofollow"
                                             target="_blank">https://doi.org/10.18103/mra.v10i7.2846</a>.
                    </li>
                    <li>Fessler J, Ting S, Yi H, Haase S, Chen J, Gulec S, et al. CytoCellDB: a comprehensive resource
                        for exploring extrachromosomal DNA in cancer cell lines. NAR Cancer 2024;6:zcae035. <a
                                href="https://doi.org/10.1093/narcan/zcae035" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1093/narcan/zcae035</a>.
                    </li>
                    <li>Fioressi SE, Bacelo DE, Duchowicz PR. Quantitative Structure–Property Relationships (QSPR) for
                        Materials Science. In: Roy K, Banerjee A, editors. Materials Informatics I, vol. 39, Cham:
                        Springer Nature Switzerland; 2025, p. 61–79. <a
                                href="https://doi.org/10.1007/978-3-031-78736-2_4" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1007/978-3-031-78736-2_4</a>.
                    </li>
                    <li>Fischer NW, Ong N, Laverty B, Psarianos P, Giovino C, Alon N, et al. TP53 variant clusters
                        stratify phenotypic diversity in germline carriers and reveal an osteosarcoma-prone subgroup.
                        Nat Commun 2025;16:8546. <a href="https://doi.org/10.1038/s41467-025-63528-6"
                                                    rel="noreferrer nofollow" target="_blank">https://doi.org/10.1038/s41467-025-63528-6</a>.
                    </li>
                    <li>Fitzpatrick MJ, Boiocchi L, Fathi AT, Brunner AM, Hasserjian RP, Nardi V. Correlation of p53
                        immunohistochemistry with TP53 mutational status and overall survival in newly diagnosed acute
                        myeloid leukaemia. Histopathology 2022;n/a. <a href="https://doi.org/10.1111/his.14726"
                                                                       rel="noreferrer nofollow" target="_blank">https://doi.org/10.1111/his.14726</a>.
                    </li>
                    <li>Fogelstrand L, Reinhardt D. Diagnosis and Classifications of Pediatric AML. In: Tomizawa D, Kolb
                        EA, Reinhardt D, editors. Acute Myeloid Leukemia in Children, Cham: Springer Nature Switzerland;
                        2024, p. 59–77. <a href="https://doi.org/10.1007/978-3-031-63366-9_4" rel="noreferrer nofollow"
                                           target="_blank">https://doi.org/10.1007/978-3-031-63366-9_4</a>.
                    </li>
                    <li>Fontecha MB, Anadón MDR, Mercado Guzmán V, Stanganelli C, Galvano C, Tosin F, et al. Genetic
                        variability profiling of the p53 signaling pathway in chronic lymphocytic leukemia. Individual
                        and combined analysis of TP53, MDM2 and NQO1 gene variants. Ann Hematol 2024. <a
                                href="https://doi.org/10.1007/s00277-024-05794-w" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1007/s00277-024-05794-w</a>.
                    </li>
                    <li>Francesca B, Meo MD, Giudice FD, Scornajenghi CM, Gazzaniga P, Berardinis ED, et al. Exploring
                        the utility of a NGS multigene panel to predict BCG response in patients with non-muscle
                        invasive bladder cancer. OR 2024;0:1–10. <a href="https://doi.org/10.32604/or.2024.056282"
                                                                    rel="noreferrer nofollow" target="_blank">https://doi.org/10.32604/or.2024.056282</a>.
                    </li>
                    <li>Funk J, Klimovich M, Pavlakis E, Neumann M, Drangenstein D, Noeparast M, et al. Functional
                        diversity of the TP53 mutome revealed by saturating CRISPR mutagenesis 2023:2023.03.10.531074.
                        <a href="https://doi.org/10.1101/2023.03.10.531074" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1101/2023.03.10.531074</a>.
                    </li>
                    <li>Funk JS, Klimovich M, Drangenstein D, Pielhoop O, Hunold P, Borowek A, et al. Deep CRISPR
                        mutagenesis characterizes the functional diversity of TP53 mutations. Nat Genet 2025;57:140–53.
                        <a href="https://doi.org/10.1038/s41588-024-02039-4" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1038/s41588-024-02039-4</a>.
                    </li>
                    <li>Funk LC, Wan J, Ryan SD, Kaur C, Sullivan R, Roopra A, et al. p53 Is Not Required for High CIN
                        to Induce Tumor Suppression. Mol Cancer Res 2021;19:112–23. <a
                                href="https://doi.org/10.1158/1541-7786.MCR-20-0488" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1158/1541-7786.MCR-20-0488</a>.
                    </li>
                    <li>Furxhi I. Health and environmental safety of nanomaterials: O Data, Where Art Thou? NanoImpact
                        2022;25:100378. <a href="https://doi.org/10.1016/j.impact.2021.100378" rel="noreferrer nofollow"
                                           target="_blank">https://doi.org/10.1016/j.impact.2021.100378</a>.
                    </li>
                    <li>Gagliano T, Kerschbamer E, Baccarani U, Minisini M, Di Giorgio E, Dalla E, et al. Changes in
                        chromatin accessibility and transcriptional landscape induced by HDAC inhibitors in TP53 mutated
                        patient-derived colon cancer organoids. Biomedicine & Pharmacotherapy 2024;173:116374. <a
                                href="https://doi.org/10.1016/j.biopha.2024.116374" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.biopha.2024.116374</a>.
                    </li>
                    <li>Gailllard B, Cornillet‐Lefebvre P, Le Q, Maloum K, Pannetier M, Lecoq‐Lafon C, et al. Clinical
                        and biological features of B‐cell neoplasms with CDK6 translocations: an association with a
                        subgroup of splenic marginal zone lymphomas displaying frequent CD5 expression, prolymphocytic
                        cells, and TP53 abnormalities. Br J Haematol 2021;193:72–82. <a
                                href="https://doi.org/10.1111/bjh.17141" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1111/bjh.17141</a>.
                    </li>
                    <li>Galante PAF, Guardia GDA, Pisani J, Sandoval RL, Barros-Filho MC, Gifoni ACLVC, et al.
                        Personalized screening strategies for TP53 R337H carriers: a retrospective cohort study of tumor
                        spectrum in Li-Fraumeni syndrome adult carriers. The Lancet Regional Health - Americas
                        2025;42:100982. <a href="https://doi.org/10.1016/j.lana.2024.100982" rel="noreferrer nofollow"
                                           target="_blank">https://doi.org/10.1016/j.lana.2024.100982</a>.
                    </li>
                    <li>Gao G, Parker J, Reynolds S, Silva T, Wang L, Al. E. Before and after: Comparison of legacy and
                        harmonized TCGA genomic data commons’ data. Cell Systems 2019;9:24-34.e10.
                    </li>
                    <li>Gao Q, Liang WW, Foltz SM, Mutharasu G, Jayasinghe RG, Cao S, et al. Driver Fusions and Their
                        Implications in the Development and Treatment of Human Cancers. Cell Reports 2018;23:227-238.e3.
                        <a href="https://doi.org/10.1016/j.celrep.2018.03.050" rel="noreferrer nofollow"
                           target="_blank">https://doi.org/10.1016/j.celrep.2018.03.050</a>.
                    </li>
                    <li>García R, Hussain A, Chen W, Wilson K, Koduru P. An artificial intelligence system applied to
                        recurrent cytogenetic aberrations and genetic progression scores predicts MYC rearrangements in
                        large B-cell lymphoma. eJHaem n.d.;n/a. <a href="https://doi.org/10.1002/jha2.451"
                                                                   rel="noreferrer nofollow" target="_blank">https://doi.org/10.1002/jha2.451</a>.
                    </li>
                    <li>García R, Timmons C, Luu H, Miller V, Fuda F, Chen W, et al. A t(11;14)(q13;q32)/CCND1::IGH
                        carrying progenitor germinal B-cell with subsequent cytogenetic aberrations contributes to the
                        development of classic Hodgkin lymphoma. Cancer Genetics 2022;268–269:97–102. <a
                                href="https://doi.org/10.1016/j.cancergen.2022.09.009" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.cancergen.2022.09.009</a>.
                    </li>
                    <li>García-Romero A, González-Arreola RM, Borjas-Gutiérrez C, Magaña-Torres MT, González-García JR.
                        Biased Clonal Evolution in Acute Promyelocytic Leukemia through Imbalances Affecting the der(17)
                        but Not the der(15) Chromosome: Report of Two Cases. CGR 2022;162:306–11. <a
                                href="https://doi.org/10.1159/000527370" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1159/000527370</a>.
                    </li>
                    <li>Garinet S, Lupo A, Denize T, Loyaux R, Timsit S, Gazeau B, et al. Successive next-generation
                        sequencing strategy for optimal fusion gene detection in non-small-cell lung cancer in clinical
                        practice. Pathology 2024;56:702–9. <a href="https://doi.org/10.1016/j.pathol.2024.02.014"
                                                              rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.pathol.2024.02.014</a>.
                    </li>
                    <li>Garsed DW, Pandey A, Fereday S, Kennedy CJ, Takahashi K, Alsop K, et al. The genomic and immune
                        landscape of long-term survivors of high-grade serous ovarian cancer. Nat Genet 2022;54:1853–64.
                        <a href="https://doi.org/10.1038/s41588-022-01230-9" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1038/s41588-022-01230-9</a>.
                    </li>
                    <li>Gazola AA, Lautert-Dutra W, Archangelo LF, Reis RBD, Squire JA. Precision oncology platforms:
                        practical strategies for genomic database utilization in cancer treatment. Mol Cytogenet
                        2024;17:28. <a href="https://doi.org/10.1186/s13039-024-00698-w" rel="noreferrer nofollow"
                                       target="_blank">https://doi.org/10.1186/s13039-024-00698-w</a>.
                    </li>
                    <li>Glusman G, Rose PW, Prlić A, Dougherty J, Duarte JM, Hoffman AS, et al. Mapping genetic
                        variations to three-dimensional protein structures to enhance variant interpretation: A proposed
                        framework. Genome Medicine 2017;9. <a href="https://doi.org/10.1186/s13073-017-0509-y"
                                                              rel="noreferrer nofollow" target="_blank">https://doi.org/10.1186/s13073-017-0509-y</a>.
                    </li>
                    <li>Golovine K, Abalakov G, Lian Z, Chatla S, Karami A, Chitrala KN, et al. ABL1 kinase as a tumor
                        suppressor in AML1-ETO and NUP98-PMX1 leukemias. Blood Cancer J 2023;13:1–7. <a
                                href="https://doi.org/10.1038/s41408-023-00810-0" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41408-023-00810-0</a>.
                    </li>
                    <li>Gopaulakrishnan S, Pollack S, Stubbs BJ, Pagès H, Readey J, Davis S, et al. Restfulse: A
                        semantically rich interface for cloud-scale genomics with bioconductor [version 1; referees: 2
                        approved]. F1000Research 2019;8. <a href="https://doi.org/10.12688/f1000research.17518.1"
                                                            rel="noreferrer nofollow" target="_blank">https://doi.org/10.12688/f1000research.17518.1</a>.
                    </li>
                    <li>Gorecki L, Muthna D, Merdita S, Andrs M, Kucera T, Havelek R, et al. 7-Azaindole,
                        2,7-diazaindole, and 1H-pyrazole as core structures for novel anticancer agents with potential
                        chemosensitizing properties. European Journal of Medicinal Chemistry 2022;240:114580. <a
                                href="https://doi.org/10.1016/j.ejmech.2022.114580" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.ejmech.2022.114580</a>.
                    </li>
                    <li>Gormley AJ. Machine learning in drug delivery. Journal of Controlled Release 2024;373:23–30. <a
                            href="https://doi.org/10.1016/j.jconrel.2024.06.045" rel="noreferrer nofollow"
                            target="_blank">https://doi.org/10.1016/j.jconrel.2024.06.045</a>.
                    </li>
                    <li>Gorunova L, Bjerkehagen B, Micci F, Heim S, Panagopoulos I. Cytogenetic and Molecular Study of
                        an Adult Sclerosing Rhabdomyosarcoma of the Extremity: MYOD1-mutation and Clonal Evolution.
                        Cancer Genomics & Proteomics 2020;17:563–9. <a href="https://doi.org/10.21873/cgp.20212"
                                                                       rel="noreferrer nofollow" target="_blank">https://doi.org/10.21873/cgp.20212</a>.
                    </li>
                    <li>Gorunova L, Boye K, Panagopoulos I, Berner J-M, Bjerkehagen B, Hompland I, et al. Cytogenetic
                        and molecular analyses of 291 gastrointestinal stromal tumors: site-specific cytogenetic
                        evolution as evidence of pathogenetic heterogeneity. Oncotarget 2022;13:508–17. <a
                                href="https://doi.org/10.18632/oncotarget.28209" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.18632/oncotarget.28209</a>.
                    </li>
                    <li>Grossman RL. Data Lakes, Clouds, and Commons: A Review of Platforms for Analyzing and Sharing
                        Genomic Data. vol. 35. 2019. <a href="https://doi.org/10.1016/j.tig.2018.12.006"
                                                        rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.tig.2018.12.006</a>.
                    </li>
                    <li>Grossman RL. Progress Toward Cancer Data Ecosystems. vol. 24. 2018. <a
                            href="https://doi.org/10.1097/PPO.0000000000000318" rel="noreferrer nofollow"
                            target="_blank">https://doi.org/10.1097/PPO.0000000000000318</a>.
                    </li>
                    <li>Gunnarsson R, Yang M, Olsson-Arvidsson L, Biloglav A, Behrendtz M, Castor A, et al. Single base
                        substitution mutational signatures in pediatric acute myeloid leukemia based on whole genome
                        sequencing. Leukemia 2021:1–5. <a href="https://doi.org/10.1038/s41375-021-01242-0"
                                                          rel="noreferrer nofollow" target="_blank">https://doi.org/10.1038/s41375-021-01242-0</a>.
                    </li>
                    <li>Haas OA. Somatic Sex: On the Origin of Neoplasms With Chromosome Counts in Uneven Ploidy Ranges.
                        Frontiers in Cell and Developmental Biology 2021;9:1886. <a
                                href="https://doi.org/10.3389/fcell.2021.631946" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.3389/fcell.2021.631946</a>.
                    </li>
                    <li>Haferlach C, Walter W, Stengel A, Meggendorfer M, Hutter S, Kern W, et al. The diverse landscape
                        of fusion transcripts in 25 different hematological entities. Leukemia & Lymphoma 2021;0:1–4. <a
                                href="https://doi.org/10.1080/10428194.2021.1953009" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1080/10428194.2021.1953009</a>.
                    </li>
                    <li>He MX, Cuoco MS, Crowdis J, Bosma-Moody A, Zhang Z, Bi K, et al. Transcriptional mediators of
                        treatment resistance in lethal prostate cancer. Nature Medicine 2021;27:426–33. <a
                                href="https://doi.org/10.1038/s41591-021-01244-6" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41591-021-01244-6</a>.
                    </li>
                    <li>Hedges DJ. RNA-seq Fusion Detection in Clinical Oncology. In: Laganà A, editor. Computational
                        Methods for Precision Oncology, Cham: Springer International Publishing; 2022, p. 163–75. <a
                                href="https://doi.org/10.1007/978-3-030-91836-1_9" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1007/978-3-030-91836-1_9</a>.
                    </li>
                    <li>Heesterbeek CJ, Aukema SM, Galjaard R-JH, Boon EMJ, Srebniak MI, Bouman K, et al. Noninvasive
                        Prenatal Test Results Indicative of Maternal Malignancies: A Nationwide Genetic and Clinical
                        Follow-Up Study. Journal of Clinical Oncology 2022;40:2426–35. <a
                                href="https://doi.org/10.1200/JCO.21.02260" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1200/JCO.21.02260</a>.
                    </li>
                    <li>Heesterbeek CJ, Lenaerts L, Tjan-Heijnen VCG, Amant F, Van Rij MC, Theunis M, et al.
                        Comprehensive Recommendations for the Clinical Management of Pregnant Women With Noninvasive
                        Prenatal Test Results Suspicious of a Maternal Malignancy. JCO Oncol Pract 2024a:OP.23.00594. <a
                                href="https://doi.org/10.1200/OP.23.00594" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1200/OP.23.00594</a>.
                    </li>
                    <li>Heesterbeek CJ, Tjan-Heijnen VCG, Heimovaara JH, Lenaerts L, Lok C, Vriens IJH, et al. Prenatal
                        cell-free DNA testing of women with pregnancy-associated cancer: a retrospective cross-sectional
                        study. The Lancet Regional Health - Europe 2024b;45:101024. <a
                                href="https://doi.org/10.1016/j.lanepe.2024.101024" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.lanepe.2024.101024</a>.
                    </li>
                    <li>Heim S, Mitelman F. Abnormal Chromosomes: The Past, Present, and Future of Cancer Cytogenetics.
                        John Wiley & Sons; 2022.
                    </li>
                    <li>Heitzeneder S, Sotillo E, Shern JF, Sindiri S, Xu P, Jones R, et al. Pregnancy-Associated Plasma
                        Protein-A (PAPP-A) in Ewing Sarcoma: Role in Tumor Growth and Immune Evasion. Journal of the
                        National Cancer Institute 2019;111:970–82. <a href="https://doi.org/10.1093/jnci/djy209"
                                                                      rel="noreferrer nofollow" target="_blank">https://doi.org/10.1093/jnci/djy209</a>.
                    </li>
                    <li>Hemminki K, Niazi Y, Vodickova L, Vodicka P, Försti A. Genetic and environmental associations of
                        nonspecific chromosomal aberrations. Mutagenesis 2024:geae006. <a
                                href="https://doi.org/10.1093/mutage/geae006" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1093/mutage/geae006</a>.
                    </li>
                    <li>Heng E, Thanedar S, Heng HH. Challenges and Opportunities for Clinical Cytogenetics in the 21st
                        Century. Genes 2023;14:493. <a href="https://doi.org/10.3390/genes14020493"
                                                       rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/genes14020493</a>.
                    </li>
                    <li>Hickey AJ, Huang C-C, Misra A, Balasubramanian V, Moudgil BM, Rosato AD, et al. Perspectives:
                        Past, Present, and Future Developments in Particle Science and Technology. KONA 2025:2026016. <a
                                href="https://doi.org/10.14356/kona.2026016" rel="noreferrer nofollow" target="_blank">https://doi.org/10.14356/kona.2026016</a>.
                    </li>
                    <li>Hindawi. Preclinical Evaluation of the Tumorigenic and Immunomodulatory Properties of Human Bone
                        Marrow Mesenchymal Stromal Cell Populations with Clonal Trisomy 5 2022.
                        https://www.hindawi.com/journals/sci/2022/1613636/ (accessed September 7, 2022).
                    </li>
                    <li>Hinkson IV, Davidsen TM, Klemm JD, Chandramouliswaran I, Kerlavage AR, Kibbe WA. A Comprehensive
                        Infrastructure for Big Data in Cancer Research: Accelerating Cancer Research and Precision
                        Medicine. Frontiers in Cell and Developmental Biology 2017;5:83. <a
                                href="https://doi.org/10.3389/fcell.2017.00083" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.3389/fcell.2017.00083</a>.
                    </li>
                    <li>Hoang V, Hung L-H, Perez D, Deng H, Schooley R, Arumilli N, et al. Container Profiler: Profiling
                        resource utilization of containerized big data pipelines. GigaScience 2023;12:giad069. <a
                                href="https://doi.org/10.1093/gigascience/giad069" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1093/gigascience/giad069</a>.
                    </li>
                    <li>Hofvander J, Jo VY, Fletcher CDM, Puls F, Flucke U, Nilsson J, et al. PHF1 fusions cause
                        distinct gene expression and chromatin accessibility profiles in ossifying fibromyxoid tumors
                        and mesenchymal cells. Modern Pathology 2020. <a
                                href="https://doi.org/10.1038/s41379-020-0457-8" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41379-020-0457-8</a>.
                    </li>
                    <li>Horie S, Saito Y, Kogure Y, Mizuno K, Ito Y, Tabata M, et al. Pan-Cancer Comparative and
                        Integrative Analyses of Driver Alterations Using Japanese and International Genomic Databases.
                        Cancer Discovery 2024:OF1–18. <a href="https://doi.org/10.1158/2159-8290.CD-23-0902"
                                                         rel="noreferrer nofollow" target="_blank">https://doi.org/10.1158/2159-8290.CD-23-0902</a>.
                    </li>
                    <li>Hosseini M-S. Current insights and future directions of Li-Fraumeni syndrome. Discov Onc
                        2024;15:561. <a href="https://doi.org/10.1007/s12672-024-01435-w" rel="noreferrer nofollow"
                                        target="_blank">https://doi.org/10.1007/s12672-024-01435-w</a>.
                    </li>
                    <li>Hoyos D, Greenbaum B, Levine AJ. The genotypes and phenotypes of missense mutations in the
                        proline domain of the p53 protein. Cell Death Differ 2022:1–8. <a
                                href="https://doi.org/10.1038/s41418-022-00980-7" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41418-022-00980-7</a>.
                    </li>
                    <li>Huang K. Multi-omics Portraits of Cancer. Dissertation 2018.</li>
                    <li>lin Huang K, Mashl RJ, Wu Y, Ritter DI, Wang J, Oh C, et al. Pathogenic Germline Variants in
                        10,389 Adult Cancers. Cell 2018;173:355-370.e14. <a
                                href="https://doi.org/10.1016/j.cell.2018.03.039" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.cell.2018.03.039</a>.
                    </li>
                    <li>Huang Q, Li F, Ji M, Lin L, Hu C. Evaluating the prognostic significance of p53 and TP53
                        mutations in HPV-negative hypopharyngeal carcinoma patients: a 5-year follow-up retrospective
                        study. BMC Cancer 2023;23:324. <a href="https://doi.org/10.1186/s12885-023-10775-9"
                                                          rel="noreferrer nofollow" target="_blank">https://doi.org/10.1186/s12885-023-10775-9</a>.
                    </li>
                    <li>Hubert J-N, Suybeng V, Vallée M, Delhomme TM, Maubec E, Boland A, et al. The PI3K/mTOR Pathway
                        Is Targeted by Rare Germline Variants in Patients with Both Melanoma and Renal Cell Carcinoma.
                        Cancers 2021;13:2243. <a href="https://doi.org/10.3390/cancers13092243"
                                                 rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/cancers13092243</a>.
                    </li>
                    <li>Hughes EG, Bergman DT, Green DC, Sukhadia SS, Lefferts JA, Karrs JX, et al. Clinical
                        Implementation of Expanded Solid Tumor Fusion Detection With Whole-Transcriptome Sequencing at
                        an Academic Cancer Center. JCO Precis Oncol 2025:e2400830. <a
                                href="https://doi.org/10.1200/PO-24-00830" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1200/PO-24-00830</a>.
                    </li>
                    <li>Hung L-H, Fukuda B, Schmitz R, Hoang V, Lloyd W, Yeung KY. Harmonizing and integrating the NCI
                        Genomic Data Commons through accessible, interactive, and cloud-enabled workflows. PLoS ONE
                        2025;20:e0318676. <a href="https://doi.org/10.1371/journal.pone.0318676"
                                             rel="noreferrer nofollow" target="_blank">https://doi.org/10.1371/journal.pone.0318676</a>.
                    </li>
                    <li>Iberti M, Loschi M, Keslair F, Deckert M, Pichery E, Di Mauro I, et al. Novel ETV6::RAPGEF6
                        fusion gene in chronic eosinophilic leukemia: compiling evidence on the role of IL3
                        overexpression in tumorigenesis. Ann Hematol 2025. <a
                                href="https://doi.org/10.1007/s00277-025-06217-0" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1007/s00277-025-06217-0</a>.
                    </li>
                    <li>Irfan M, Fatima U, Sajjad A, Naveed M. Nanoinformatics: A Potential Tool for Precision Targeting
                        in Pediatric Cancer. BioNanoSci 2025;15:516. <a
                                href="https://doi.org/10.1007/s12668-025-02126-8" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1007/s12668-025-02126-8</a>.
                    </li>
                    <li>Jamur V, Lisboa M, De Azambuja A, Brofman P. Case report: CD8 positive T-cell Prolymphocytic
                        Leukemia presenting complex karyotype with a rare chromosomal abnormality der(2)t(1;2)(q21;q37)
                        and additional signals of the genes MYC, IGH, and TP53. Hematology, Transfusion and Cell Therapy
                        2023. <a href="https://doi.org/10.1016/j.htct.2023.07.009" rel="noreferrer nofollow"
                                 target="_blank">https://doi.org/10.1016/j.htct.2023.07.009</a>.
                    </li>
                    <li>Jeznach O, Tabakoglu S, Zaszczyńska A, Sajkiewicz P. Review on machine learning application in
                        tissue engineering: What has been done so far? Application areas, challenges, and perspectives.
                        J Mater Sci 2024. <a href="https://doi.org/10.1007/s10853-024-10449-2" rel="noreferrer nofollow"
                                             target="_blank">https://doi.org/10.1007/s10853-024-10449-2</a>.
                    </li>
                    <li>Joyce P, Allen CJ, Alonso MJ, Ashford M, Bradbury MS, Germain M, et al. A translational
                        framework to DELIVER nanomedicines to the clinic. Nat Nanotechnol 2024. <a
                                href="https://doi.org/10.1038/s41565-024-01754-7" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41565-024-01754-7</a>.
                    </li>
                    <li>Kanaoka D, Yamada M, Yokoyama H, Nishino S, Kunimura N, Satoyoshi H, et al. FPFT-2216, a Novel
                        Anti-lymphoma Compound, Induces Simultaneous Degradation of IKZF1/3 and CK1α to Activate p53 and
                        Inhibit NFκB Signaling. Cancer Research Communications 2024;4:312–27. <a
                                href="https://doi.org/10.1158/2767-9764.CRC-23-0264" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1158/2767-9764.CRC-23-0264</a>.
                    </li>
                    <li>Kannan M. Artificial Intelligence in Cancer Researches. International Journal of Trends in
                        OncoScience 2023:19–26.
                    </li>
                    <li>Kao YC, Lee JC. An update of molecular findings in uterine tumor resembling ovarian sex cord
                        tumor and GREB1-rearranged uterine sarcoma with variable sex-cord differentiation. Blackwell
                        Publishing Inc.; 2020. <a href="https://doi.org/10.1002/gcc.22909" rel="noreferrer nofollow"
                                                  target="_blank">https://doi.org/10.1002/gcc.22909</a>.
                    </li>
                    <li>Karlebach G, Veiga DFT, Mays AD, Kesarwani AK, Danis D, Kararigas G, et al. The impact of sex on
                        alternative splicing. bioRxiv 2018:490904. <a href="https://doi.org/10.1101/490904"
                                                                      rel="noreferrer nofollow" target="_blank">https://doi.org/10.1101/490904</a>.
                    </li>
                    <li>Kasperski A, Heng HH. The Digital World of Cytogenetic and Cytogenomic Web Resources. In: Ye JC,
                        Heng HH, editors. Cancer Cytogenetics and Cytogenomics, vol. 2825, New York, NY: Springer US;
                        2024, p. 361–91. <a href="https://doi.org/10.1007/978-1-0716-3946-7_21"
                                            rel="noreferrer nofollow" target="_blank">https://doi.org/10.1007/978-1-0716-3946-7_21</a>.
                    </li>
                    <li>Ke W, Crist RM, Clogston JD, Stern ST, Dobrovolskaia MA, Grodzinski P, et al. Trends and
                        patterns in cancer nanotechnology research: A survey of NCI’s caNanoLab and nanotechnology
                        characterization laboratory. Advanced Drug Delivery Reviews 2022;191:114591. <a
                                href="https://doi.org/10.1016/j.addr.2022.114591" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.addr.2022.114591</a>.
                    </li>
                    <li>Ke W, He R, Jensen MA, Dobrovolskaia MA. Transforming Cancer Nanotechnology Data Analysis and
                        User Experience. Part I: Current Challenges and Solutions Provided by CANANOLAB. WIREs Nanomed
                        Nanobiotechnol 2025;17:e70030. <a href="https://doi.org/10.1002/wnan.70030"
                                                          rel="noreferrer nofollow" target="_blank">https://doi.org/10.1002/wnan.70030</a>.
                    </li>
                    <li>Keimakh D. Robust Identification of Gene Fusions from RNA Sequencing Data and Their Potential as
                        Biomarkers of Drug Response. Thesis. 2021.
                    </li>
                    <li>Kent MR, Silvius K, Kucinski J, Calderon D, Kendall GC. Functional Genomics of Novel
                        Rhabdomyosarcoma Fusion-Oncogenes Using Zebrafish. In: Amatruda JF, Houart C, Kawakami K, Poss
                        KD, editors. Zebrafish: Methods and Protocols, New York, NY: Springer US; 2024, p. 23–41. <a
                                href="https://doi.org/10.1007/978-1-0716-3401-1_2" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1007/978-1-0716-3401-1_2</a>.
                    </li>
                    <li>Keshavarz-Rahaghi F, Pleasance E, Kolisnik T, Jones SJM. A p53 transcriptional signature in
                        primary and metastatic cancers derived using machine learning. Frontiers in Genetics 2022;13.
                    </li>
                    <li>Khan AA, Khan TA, Omri A. Computational Designing in Nanomedicine. In: Aziz MA, editor.
                        Personalized and Precision Nanomedicine for Cancer Treatment, Singapore: Springer Nature
                        Singapore; 2024, p. 247–75. <a href="https://doi.org/10.1007/978-981-97-3545-7_11"
                                                       rel="noreferrer nofollow" target="_blank">https://doi.org/10.1007/978-981-97-3545-7_11</a>.
                    </li>
                    <li>Khan F, Akhtar S, Kamal MA. Nanoinformatics and Personalized Medicine: An Advanced Cumulative
                        Approach for Cancer Management. Current Medicinal Chemistry 2023;30:271–85. <a
                                href="https://doi.org/10.2174/0929867329666220610090405" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.2174/0929867329666220610090405</a>.
                    </li>
                    <li>Khan S, Tang Y, Song Z, Chen X, Wang L, Zhang C, et al. Beyond chromosomal rearrangements: The
                        expanding landscape of gene fusions and chimeric RNAs. GPD 2024;0:3641. <a
                                href="https://doi.org/10.36922/gpd.3641" rel="noreferrer nofollow" target="_blank">https://doi.org/10.36922/gpd.3641</a>.
                    </li>
                    <li>Kim E, Davidsen TM, Davis-Dusenbery B, Baumann A, Maggio A, Chen Z, et al. NCI Cancer Research
                        Data Commons: Lessons Learned and Future State. Cancer Research 2024. <a
                                href="https://doi.org/10.1158/0008-5472.CAN-23-2730" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1158/0008-5472.CAN-23-2730</a>.
                    </li>
                    <li>Kim H, Kim S, Wade T, Yeo E, Lipsa A, Golebiewska A, et al. Mapping extrachromosomal DNA
                        amplifications during cancer progression. Nat Genet 2024. <a
                                href="https://doi.org/10.1038/s41588-024-01949-7" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41588-024-01949-7</a>.
                    </li>
                    <li>Kim H, Nguyen N, Turner K, Wu S, Liu J, Deshpande V, et al. Frequent extrachromosomal oncogene
                        amplification drives aggressive tumors. bioRxiv 2019:859306. <a
                                href="https://doi.org/10.1101/859306" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1101/859306</a>.
                    </li>
                    <li>Kim H, Nguyen NP, Turner K, Wu S, Gujar AD, Luebeck J, et al. Extrachromosomal DNA is associated
                        with oncogene amplification and poor outcome across multiple cancers. Nature Genetics 2020:1–7.
                        <a href="https://doi.org/10.1038/s41588-020-0678-2" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1038/s41588-020-0678-2</a>.
                    </li>
                    <li>Kim JH, Megquier K, Thomas R, Sarver AL, Song JM, Kim YT, et al. Genomically Complex Human
                        Angiosarcoma and Canine Hemangiosarcoma Establish Convergent Angiogenic Transcriptional Programs
                        Driven by Novel Gene Fusions. Mol Cancer Res 2021;19:847–61. <a
                                href="https://doi.org/10.1158/1541-7786.MCR-20-0937" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1158/1541-7786.MCR-20-0937</a>.
                    </li>
                    <li>Kim J-Y, Park K, Park W-Y, Ahn JS, Im Y-H, Lee JE, et al. Prognostic value of structural
                        variants in early breast cancer patients. Npj Breast Cancer 2024;10:64. <a
                                href="https://doi.org/10.1038/s41523-024-00669-9" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41523-024-00669-9</a>.
                    </li>
                    <li>Kim M, Jin L, Jin H-Y, Cho N-Y, Han S, Kim T-Y, et al. Aberrant Cytoplasmic p53 Expression and
                        Its Correlation with TP53 Mutation Status and Functional Implications in Stage II and III
                        Colorectal Cancer. Pathobiology 2025:1–9. <a href="https://doi.org/10.1159/000548104"
                                                                     rel="noreferrer nofollow" target="_blank">https://doi.org/10.1159/000548104</a>.
                    </li>
                    <li>Kjønnøy IJC. Characterization of Neoplasm-Specific Chromosomal Aberrations and Their Molecular
                        Consequences in Lipomas: Identification of the HMGA2::LINC00535 Fusion Transcript. Master
                        thesis. Norwegian University of Life Sciences, 2024.
                    </li>
                    <li>Kleijn TG, Ameline B, Schreuder WH, Kooistra W, Doff JJ, Witjes M, et al. Odontogenic Myxomas
                        Harbor Recurrent Copy Number Alterations and a Distinct Methylation Signature. American Journal
                        of Surgical Pathology 2024;48:1224–32. <a href="https://doi.org/10.1097/PAS.0000000000002293"
                                                                  rel="noreferrer nofollow" target="_blank">https://doi.org/10.1097/PAS.0000000000002293</a>.
                    </li>
                    <li>Klett T, Stahlecker J, Jaag S, Masberg B, Knappe C, Lämmerhofer M, et al. Covalent Fragments
                        Acting as Tyrosine Mimics for Mutant p53-Y220C Rescue by Nucleophilic Aromatic Substitution. ACS
                        Pharmacol Transl Sci 2024:acsptsci.4c00414. <a href="https://doi.org/10.1021/acsptsci.4c00414"
                                                                       rel="noreferrer nofollow" target="_blank">https://doi.org/10.1021/acsptsci.4c00414</a>.
                    </li>
                    <li>Klockner TC, Campbell CS. Selection forces underlying aneuploidy patterns in cancer. Molecular &
                        Cellular Oncology 2024;11:2369388. <a href="https://doi.org/10.1080/23723556.2024.2369388"
                                                              rel="noreferrer nofollow" target="_blank">https://doi.org/10.1080/23723556.2024.2369388</a>.
                    </li>
                    <li>Knott MML, Hölting TLB, Ohmura S, Kirchner T, Cidre-Aranaz F, Grünewald TGP. Targeting the
                        undruggable: exploiting neomorphic features of fusion oncoproteins in childhood sarcomas for
                        innovative therapies. vol. 38. Springer; 2019. <a
                                href="https://doi.org/10.1007/s10555-019-09839-9" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1007/s10555-019-09839-9</a>.
                    </li>
                    <li>Ko C, Brody JP. A genetic risk score for glioblastoma multiforme based on copy number
                        variations. Cancer Treatment and Research Communications 2021;27:100352. <a
                                href="https://doi.org/10.1016/j.ctarc.2021.100352" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.ctarc.2021.100352</a>.
                    </li>
                    <li>Koppad S, B A, Gkoutos GV, Acharjee A. Cloud Computing Enabled Big Multi-Omics Data Analytics.
                        Bioinform Biol Insights 2021;15:11779322211035921. <a
                                href="https://doi.org/10.1177/11779322211035921" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1177/11779322211035921</a>.
                    </li>
                    <li>Kosik P, Skorvaga M, Belyaev I. Preleukemic Fusion Genes Induced via Ionizing Radiation.
                        International Journal of Molecular Sciences 2023;24:6580. <a
                                href="https://doi.org/10.3390/ijms24076580" rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/ijms24076580</a>.
                    </li>
                    <li>Köster J, Arbajian E, Viklund B, Isaksson A, Hofvander J, Haglund F, et al. Genomic and
                        transcriptomic features of dermatofibrosarcoma protuberans: Unusual chromosomal origin of the
                        COL1A1-PDGFB fusion gene and synergistic effects of amplified regions in tumor development.
                        Cancer Genetics 2020;241:34–41. <a href="https://doi.org/10.1016/j.cancergen.2019.12.001"
                                                           rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.cancergen.2019.12.001</a>.
                    </li>
                    <li>Kou SH, Li J, Tam B, Lei H, Zhao B, Xiao F, et al. TP53 germline pathogenic variants in modern
                        humans were likely originated during recent human history. NAR Cancer 2023;5:zcad025. <a
                                href="https://doi.org/10.1093/narcan/zcad025" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1093/narcan/zcad025</a>.
                    </li>
                    <li>Koudjiwan P, Anderson B, Vitalis B, Katele M, Hser E, Jackson G, et al. Predicting Neoantigens
                        for Cancer Using Next-Generation IEDB & CEDAR Tools 2024. <a
                                href="https://doi.org/10.5281/ZENODO.13345145" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.5281/ZENODO.13345145</a>.
                    </li>
                    <li>Kumar A. Chemical Threats to Nature: Life 2022. <a href="https://doi.org/10.5281/ZENODO.6819272"
                                                                           rel="noreferrer nofollow" target="_blank">https://doi.org/10.5281/ZENODO.6819272</a>.
                    </li>
                    <li>Landsburg DJ, Morrissette JJ, Nasta SD, Barta SK, Schuster SJ, Svoboda J, et al. TP53 Mutations
                        Detected By Clinical Laboratory Mutation Analysis Predict for Inferior Clinical Outcomes in
                        Patients with Newly-Diagnosed Aggressive B Cell Lymphomas, Including Those with High-Risk
                        Features. Blood 2023a;142:1739. <a href="https://doi.org/10.1182/blood-2023-186343"
                                                           rel="noreferrer nofollow" target="_blank">https://doi.org/10.1182/blood-2023-186343</a>.
                    </li>
                    <li>Landsburg DJ, Morrissette JJ, Nasta SD, Barta SK, Schuster SJ, Svoboda J, et al. TP53 mutations
                        predict for poor outcomes in patients with newly diagnosed aggressive B-cell lymphomas in the
                        current era. Blood Advances 2023b;7:7243–53. <a
                                href="https://doi.org/10.1182/bloodadvances.2023011384" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1182/bloodadvances.2023011384</a>.
                    </li>
                    <li>Langmead B, Nellore A. Cloud computing for genomic data analysis and collaboration. vol. 19.
                        2018. <a href="https://doi.org/10.1038/nrg.2017.113" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1038/nrg.2017.113</a>.
                    </li>
                    <li>Lanic M-D, Guérin R, Wassef M, Durdilly P, Rainville V, Sater V, et al. Detection of salivary
                        gland and sinonasal fusions by a next-generation sequencing based, ligation-dependent, multiplex
                        RT-PCR assay. Histopathology 2023;n/a. <a href="https://doi.org/10.1111/his.14971"
                                                                  rel="noreferrer nofollow" target="_blank">https://doi.org/10.1111/his.14971</a>.
                    </li>
                    <li>Lasota J, Krupińska M, Kaczorowski M, Chłopek M, Kinkor Z, Švajdler M, et al. Utility of Protein
                        Kinase C Beta II Immunohistochemistry in Differential Diagnosis of Ewing Sarcoma. American
                        Journal of Surgical Pathology 2025. <a href="https://doi.org/10.1097/PAS.0000000000002400"
                                                               rel="noreferrer nofollow" target="_blank">https://doi.org/10.1097/PAS.0000000000002400</a>.
                    </li>
                    <li>Laurent AP, Kotecha RS, Malinge S. Gain of chromosome 21 in hematological malignancies: lessons
                        from studying leukemia in children with Down syndrome. Leukemia 2020:1–16. <a
                                href="https://doi.org/10.1038/s41375-020-0854-5" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41375-020-0854-5</a>.
                    </li>
                    <li>Lauricella C, Greco R, Mancini V, Motta V, Ciraolo A, Canal GD, et al. Acute Promyelocytic
                        Leukemia with del(6)(p22) and Atypical bcr2 PML::RARA Fusion Transcript: A Case Report. AHA
                        2022:1–7. <a href="https://doi.org/10.1159/000527316" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1159/000527316</a>.
                    </li>
                    <li>Lazarou G, Chelliah V, Small BG, Walker M, van der Graaf PH, Kierzek AM. Integration of Omics
                        Data Sources to Inform Mechanistic Modeling of Immune-Oncology Therapies: A Tutorial for
                        Clinical Pharmacologists. Clinical Pharmacology and Therapeutics 2020;107:858–70. <a
                                href="https://doi.org/10.1002/cpt.1786" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1002/cpt.1786</a>.
                    </li>
                    <li>Le Duc D, Lin C-C, Popkova Y, Yang Z, Akhil V, Çakir MV, et al. Reduced lipolysis in lipoma
                        phenocopies lipid accumulation in obesity. International Journal of Obesity 2020. <a
                                href="https://doi.org/10.1038/s41366-020-00716-y" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41366-020-00716-y</a>.
                    </li>
                    <li>Leclair NK, Brugiolo M, Park S, Devoucoux M, Urbanski L, Angarola BL, et al. Antisense
                        oligonucleotide-mediated TRA2β poison exon inclusion induces the expression of a lncRNA with
                        anti-tumor effects. Nat Commun 2025;16:1670. <a
                                href="https://doi.org/10.1038/s41467-025-56913-8" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41467-025-56913-8</a>.
                    </li>
                    <li>Leclair NK, Brugiolo M, Urbanski L, Lawson SC, Thakar K, Yurieva M, et al. Poison Exon Splicing
                        Regulates a Coordinated Network of SR Protein Expression during Differentiation and
                        Tumorigenesis. Molecular Cell 2020;80:648-665.e9. <a
                                href="https://doi.org/10.1016/j.molcel.2020.10.019" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.molcel.2020.10.019</a>.
                    </li>
                    <li>Lehmann J, Aly A, Steffke C, Fabbio L, Mayer V, Dikwella N, et al. Heterozygous knockout of
                        Synaptotagmin13 phenocopies ALS features and TP53 activation in human motor neurons. Cell Death
                        Dis 2024;15:560. <a href="https://doi.org/10.1038/s41419-024-06957-3" rel="noreferrer nofollow"
                                            target="_blank">https://doi.org/10.1038/s41419-024-06957-3</a>.
                    </li>
                    <li>Lejman M, Chałupnik A, Chilimoniuk Z, Dobosz M. Genetic Biomarkers and Their Clinical
                        Implications in B-Cell Acute Lymphoblastic Leukemia in Children. International Journal of
                        Molecular Sciences 2022;23:2755. <a href="https://doi.org/10.3390/ijms23052755"
                                                            rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/ijms23052755</a>.
                    </li>
                    <li>Li GXL, Munro D, Fermin D, Vogel C, Choi H. A protein-centric approach for exome variant
                        aggregation enables sensitive association analysis with clinical outcomes. Human Mutation
                        2020;41:934–45. <a href="https://doi.org/10.1002/humu.23979" rel="noreferrer nofollow"
                                           target="_blank">https://doi.org/10.1002/humu.23979</a>.
                    </li>
                    <li>Li H, Wang Q. Chimeric RNAs and their implication in prostate cancer. Cancer Pathogenesis and
                        Therapy 2023. <a href="https://doi.org/10.1016/j.cpt.2023.04.003" rel="noreferrer nofollow"
                                         target="_blank">https://doi.org/10.1016/j.cpt.2023.04.003</a>.
                    </li>
                    <li>Li M, Li Q-Z, Zhao Y, Gao X. Recent Advances in Machine Learning Models for Predicting Toxicity
                        of Inorganic Nanoparticles. Chem Bio Eng 2025:cbe.5c00048. <a
                                href="https://doi.org/10.1021/cbe.5c00048" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1021/cbe.5c00048</a>.
                    </li>
                    <li>Liang F, Zhang Y, Zhou C, Zhang H, Liu G, Zhu J. Segmentation study of nanoparticle topological
                        structures based on synthetic data. PLoS ONE 2024;19:e0311228. <a
                                href="https://doi.org/10.1371/journal.pone.0311228" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1371/journal.pone.0311228</a>.
                    </li>
                    <li>Lilljebjörn H, Orsmark-Pietras C, Mitelman F, Hagström-Andersson A, Fioretos T. Transcriptomics
                        paving the way for improved diagnostics and precision medicine of acute leukemia. Seminars in
                        Cancer Biology 2021. <a href="https://doi.org/10.1016/j.semcancer.2021.09.013"
                                                rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.semcancer.2021.09.013</a>.
                    </li>
                    <li>Lim HJ, Lee JH, Lee SY, Choi H-W, Choi H-J, Kee S-J, et al. Diagnostic Validation of a Clinical
                        Laboratory-Oriented Targeted RNA Sequencing System for Detecting Gene Fusions in Hematologic
                        Malignancies. The Journal of Molecular Diagnostics 2021. <a
                                href="https://doi.org/10.1016/j.jmoldx.2021.05.008" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.jmoldx.2021.05.008</a>.
                    </li>
                    <li>Lin Z, Zhang Z, Chen C-Y, Chen Q, Wu P-Y. Application of machine learning and artificial
                        intelligence methods in toxicity assessment of nanoparticles. Machine Learning and Artificial
                        Intelligence in Toxicology and Environmental Health, Elsevier; 2026, p. 197–232. <a
                                href="https://doi.org/10.1016/B978-0-443-30010-3.00007-6" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/B978-0-443-30010-3.00007-6</a>.
                    </li>
                    <li>Liu-Smith F, Lu Y. Opposite Roles of BAP1 in Overall Survival of Uveal Melanoma and Cutaneous
                        Melanoma. Journal of Clinical Medicine 2020;9:411. <a href="https://doi.org/10.3390/jcm9020411"
                                                                              rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/jcm9020411</a>.
                    </li>
                    <li>Lo CS. Functional Assessment on the Differential Effect of Mutant p53-Plakoglobin Interaction in
                        Carcinoma Cells. ERA 2023. <a href="https://doi.org/10.7939/r3-b12y-7833"
                                                      rel="noreferrer nofollow" target="_blank">https://doi.org/10.7939/r3-b12y-7833</a>.
                    </li>
                    <li>Lomov N, Zerkalenkova E, Lebedeva S, Viushkov V, Rubtsov MA. Cytogenetic and molecular genetic
                        methods for chromosomal translocations detection with reference to the KMT2A/MLL gene. Taylor
                        and Francis Ltd.; 2020. <a href="https://doi.org/10.1080/10408363.2020.1844135"
                                                   rel="noreferrer nofollow" target="_blank">https://doi.org/10.1080/10408363.2020.1844135</a>.
                    </li>
                    <li>Lopez de Lapuente Portilla A, Ekdahl L, Cafaro C, Ali Z, Miharada N, Thorleifsson G, et al.
                        Genome-wide association study on 13 167 individuals identifies regulators of blood CD34+cell
                        levels. Blood 2022;139:1659–69. <a href="https://doi.org/10.1182/blood.2021013220"
                                                           rel="noreferrer nofollow" target="_blank">https://doi.org/10.1182/blood.2021013220</a>.
                    </li>
                    <li>Luebeck J, Ng AWT, Galipeau PC, Li X, Sanchez CA, Katz-Summercorn AC, et al. Extrachromosomal
                        DNA in the cancerous transformation of Barrett’s oesophagus. Nature 2023;616:798–805. <a
                                href="https://doi.org/10.1038/s41586-023-05937-5" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41586-023-05937-5</a>.
                    </li>
                    <li>Lundin-Ström KB, Biloglav A, Lazarevic V, Behrendtz M, Castor A, Johansson B. Parental origin of
                        monosomy 7 in acute leukaemia. Br J Haematol 2021;192. <a
                                href="https://doi.org/10.1111/bjh.17356" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1111/bjh.17356</a>.
                    </li>
                    <li>Ma X, Mao M, He J, Liang C, Xie H-Y. Nanoprobe-based molecular imaging for tumor stratification.
                        Chem Soc Rev 2023. <a href="https://doi.org/10.1039/D3CS00063J" rel="noreferrer nofollow"
                                              target="_blank">https://doi.org/10.1039/D3CS00063J</a>.
                    </li>
                    <li>Macagno N, Pissaloux D, de la Fouchardière A, Karanian M, Lantuejoul S, Galateau Salle F, et al.
                        Wholistic approach: Transcriptomic analysis and beyond using archival material for molecular
                        diagnosis. Genes, Chromosomes and Cancer n.d.;n/a. <a href="https://doi.org/10.1002/gcc.23026"
                                                                              rel="noreferrer nofollow" target="_blank">https://doi.org/10.1002/gcc.23026</a>.
                    </li>
                    <li>Madeddu C, Lai E, Neri M, Sanna E, Gramignano G, Nemolato S, et al. Association Between TP53
                        Mutations and Platinum Resistance in a Cohort of High-Grade Serous Ovarian Cancer Patients:
                        Novel Implications for Personalized Therapeutics. IJMS 2025;26:2232. <a
                                href="https://doi.org/10.3390/ijms26052232" rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/ijms26052232</a>.
                    </li>
                    <li>Maitre E, Tomowiak C, Lebecque B, Bijou F, Benabed K, Naguib D, et al. Deciphering Genetic
                        Alterations of Hairy Cell Leukemia and Hairy Cell Leukemia-like Disorders in 98 Patients.
                        Cancers (Basel) 2022;14:1904. <a href="https://doi.org/10.3390/cancers14081904"
                                                         rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/cancers14081904</a>.
                    </li>
                    <li>Majhi P, Sharma A, Jerry DJ. Genetic modifiers of p53: opportunities for breast cancer
                        therapies. Oncotarget 2023;14:236–41. <a href="https://doi.org/10.18632/oncotarget.28387"
                                                                 rel="noreferrer nofollow" target="_blank">https://doi.org/10.18632/oncotarget.28387</a>.
                    </li>
                    <li>Malcikova J, Pavlova S, Baliakas P, Chatzikonstantinou T, Tausch E, Catherwood M, et al. ERIC
                        recommendations for TP53 mutation analysis in chronic lymphocytic leukemia—2024 update. Leukemia
                        2024. <a href="https://doi.org/10.1038/s41375-024-02267-x" rel="noreferrer nofollow"
                                 target="_blank">https://doi.org/10.1038/s41375-024-02267-x</a>.
                    </li>
                    <li>Malhotra L, Singh A, Kaur P, Ethayathulla AS. Phenotypical mapping of TP53 unique missense
                        mutations spectrum in human cancers 2023:2023.02.06.527255. <a
                                href="https://doi.org/10.1101/2023.02.06.527255" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1101/2023.02.06.527255</a>.
                    </li>
                    <li>Malhotra L, Singh A, Kaur P, Ethayathulla AS. Comprehensive omics studies of p53 mutants in
                        human cancer. Briefings in Functional Genomics 2022:elac015. <a
                                href="https://doi.org/10.1093/bfgp/elac015" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1093/bfgp/elac015</a>.
                    </li>
                    <li>Manabe M, Inano N, Hagiwara Y, Koh K-R. Diffuse Large B-cell Lymphoma with a Multilobated
                        Nuclear Morphology Harboring t (14;18) (q32;q21) and t (3;22) (q27;q11): A Case Report. Intern
                        Med 2025:4719–24. <a href="https://doi.org/10.2169/internalmedicine.4719-24"
                                             rel="noreferrer nofollow" target="_blank">https://doi.org/10.2169/internalmedicine.4719-24</a>.
                    </li>
                    <li>Mandahl N, Mitelman F. Giemsa-negative chromosome bands preferentially recombine in
                        cancer-associated translocations and gene fusions. Genes, Chromosomes and Cancer 2022;n/a. <a
                                href="https://doi.org/10.1002/gcc.23095" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1002/gcc.23095</a>.
                    </li>
                    <li>Mansur MB, deSouza NM, Natrajan R, Abegglen LM, Schiffman JD, Greaves M. Evolutionary
                        determinants of curability in cancer. Nat Ecol Evol 2023:1–10. <a
                                href="https://doi.org/10.1038/s41559-023-02159-w" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41559-023-02159-w</a>.
                    </li>
                    <li>Mansur MB, Greaves M. Convergent TP53 loss and evolvability in cancer. BMC Ecol Evo 2023;23:54.
                        <a href="https://doi.org/10.1186/s12862-023-02146-6" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1186/s12862-023-02146-6</a>.
                    </li>
                    <li>Marco-Brualla J, de Miguel D, Martínez-Lostao L, Anel A. DR5 Up-Regulation Induced by
                        Dichloroacetate Sensitizes Tumor Cells to Lipid Nanoparticles Decorated with TRAIL. Journal of
                        Clinical Medicine 2023;12:608. <a href="https://doi.org/10.3390/jcm12020608"
                                                          rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/jcm12020608</a>.
                    </li>
                    <li>Martinelli C, Gabriele F, Manai F, Ciccone R, Novara F, Sauta E, et al. The search for molecular
                        markers in a gene-orphan case study of a pediatric spinal cord pilocytic astrocytoma. Cancer
                        Genomics and Proteomics 2020;17:117–30. <a href="https://doi.org/10.21873/cgp.20172"
                                                                   rel="noreferrer nofollow" target="_blank">https://doi.org/10.21873/cgp.20172</a>.
                    </li>
                    <li>Martins-Chaves RR, Silva RSO, Pereira T dos SF, Fonseca FP, Gomez RS. Evolution tumour models
                        paving the way for understanding oral carcinogenesis. Journal of Oral Pathology & Medicine
                        2023;n/a. <a href="https://doi.org/10.1111/jop.13402" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1111/jop.13402</a>.
                    </li>
                    <li>Matsukawa T, Aplan PD. Clinical and Molecular Consequences of Fusion Genes in Myeloid
                        Malignancies. STEM CELLS 2020. <a href="https://doi.org/10.1002/stem.3263"
                                                          rel="noreferrer nofollow" target="_blank">https://doi.org/10.1002/stem.3263</a>.
                    </li>
                    <li>Matsumoto N, Manrai P, Rottmann D, Wu X, Assem H, Hui P, et al. Correlative Assessment of p53
                        Immunostaining Patterns and TP53 Mutation Status by Next-Generation Sequencing in High-Grade
                        Endometrial Carcinomas. International Journal of Gynecological Pathology
                        n.d.:10.1097/PGP.0000000000000930. <a href="https://doi.org/10.1097/PGP.0000000000000930"
                                                              rel="noreferrer nofollow" target="_blank">https://doi.org/10.1097/PGP.0000000000000930</a>.
                    </li>
                    <li>MEDISCHE LABORATORIA DIE NEXT GENERATION SEQUENCING (NGS) ANALYSEN VOOR HEMATOLOGISCHE EN SOLIDE
                        TUMOREN UITVOEREN. NGS Guidelines 2021.
                    </li>
                    <li>Meggendorfer M, Walter W, Haferlach T. WGS and WTS in leukaemia: A tool for diagnostics? Best
                        Practice & Research Clinical Haematology 2020:101190. <a
                                href="https://doi.org/10.1016/j.beha.2020.101190" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.beha.2020.101190</a>.
                    </li>
                    <li>Mešanović S, Krašić A. Analysis of cytogenetic aberrations in a bone marrow of patients with
                        malignant head and neck lymphadenopathies - A single center experience. Genetics & Applications
                        2022;6:70–8. <a href="https://doi.org/10.31383/ga.vol6iss2ga07" rel="noreferrer nofollow"
                                        target="_blank">https://doi.org/10.31383/ga.vol6iss2ga07</a>.
                    </li>
                    <li>Micheloni G, Frattini A, Donini M, Dusi S, Leszl A, Di Meglio A, et al. Donor Cell Acute Myeloid
                        Leukemia after Hematopoietic Stem Cell Transplantation for Chronic Granulomatous Disease: A Case
                        Report and Literature Review. Genes 2023;14:2085. <a
                                href="https://doi.org/10.3390/genes14112085" rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/genes14112085</a>.
                    </li>
                    <li>Miga KH, Sullivan BA. Expanding studies of chromosome structure and function in the era of T2T
                        genomics. Human Molecular Genetics 2021. <a href="https://doi.org/10.1093/hmg/ddab214"
                                                                    rel="noreferrer nofollow" target="_blank">https://doi.org/10.1093/hmg/ddab214</a>.
                    </li>
                    <li>Mikkilineni S, Pineda-Reyes JP, Wilde L, Ferber A, Wang Z-X, Peiper S, et al. Case Report:
                        Myeloid neoplasms with the t(3;12)(q26.2;p13.1)/MECOM-ETV6 translocation: report of two new
                        cases and review of the literature. Front Oncol 2025;15:1526044. <a
                                href="https://doi.org/10.3389/fonc.2025.1526044" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.3389/fonc.2025.1526044</a>.
                    </li>
                    <li>Miller, Konstatine; Heslop-Harrison, J.S; Madan, Kamlesh; Rocchi M. Mitelman Database of
                        Chromosome Aberrations and Gene Fusions in Cancer. European Cytogenetics Association Newsletter
                        2020;46:19.
                    </li>
                    <li>Mirgayazova R, Khadiullina R, Filimonova M, Chasov V, Bulatov E. Impact of TP53 mutations on the
                        efficacy of CAR-T cell therapy in cancer. Explor Immunol 2024:837–52. <a
                                href="https://doi.org/10.37349/ei.2024.00176" rel="noreferrer nofollow" target="_blank">https://doi.org/10.37349/ei.2024.00176</a>.
                    </li>
                    <li>Mirkin CA, Petrosko SH, Artzi N, Aydin K, Biaggne A, Brinker CJ, et al. 33 Unresolved Questions
                        in Nanoscience and Nanotechnology. ACS Nano 2025;19:31933–68. <a
                                href="https://doi.org/10.1021/acsnano.5c12854" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1021/acsnano.5c12854</a>.
                    </li>
                    <li>Mirzaei G. GraphChrom: A Novel Graph-Based Framework for Cancer Classification Using Chromosomal
                        Rearrangement Endpoints. Cancers 2022;14:3060. <a href="https://doi.org/10.3390/cancers14133060"
                                                                          rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/cancers14133060</a>.
                    </li>
                    <li>Mischel P. Extrachromosomal DNA is associated with oncogene amplification and poor outcome
                        across multiple cancers | Nature Research Cancer Community. 2020.
                    </li>
                    <li>Mishra V, Pandey RP, Priyadarshini A, Chang C-M, Leal E, editors. Nanotherapeutics for
                        inflammatory arthritis: design, diagnosis, and treatment. First edition. Boca Raton, FL: CRC
                        Press; 2024.
                    </li>
                    <li>Mitelman F, Mandahl N. Distinct Signatures of Chromosomal Involvement in 59 251 Translocations
                        Across 58 Tumor Types. A Novel Perspective. Genes Chromosomes &amp; Cancer 2025;64:e70053. <a
                                href="https://doi.org/10.1002/gcc.70053" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1002/gcc.70053</a>.
                    </li>
                    <li>Mocellin S. Etiology and Predisposition. Soft Tissue Tumors, Cham: Springer International
                        Publishing; 2021, p. 11–5. <a href="https://doi.org/10.1007/978-3-030-58710-9_3"
                                                      rel="noreferrer nofollow" target="_blank">https://doi.org/10.1007/978-3-030-58710-9_3</a>.
                    </li>
                    <li>Mochão HDC. Improvement of KiMoSys framework for kinetic modelling 2021.</li>
                    <li>Mohamed W, editor. Toward Applicability of Exosomal-ncRNA Precision vs. Nanoparticles to the
                        Brain. Nanocarriers in neurodegenerative disorders: therapeutic hopes and hypes. First edition,
                        Boca Raton FL: CRC Press; 2024.
                    </li>
                    <li>Möller E, Praz V, Rajendran S, Dong R, Cauderay A, Xing Y-H, et al. EWSR1-ATF1 dependent 3D
                        connectivity regulates oncogenic and differentiation programs in Clear Cell Sarcoma. Nat Commun
                        2022;13:2267. <a href="https://doi.org/10.1038/s41467-022-29910-4" rel="noreferrer nofollow"
                                         target="_blank">https://doi.org/10.1038/s41467-022-29910-4</a>.
                    </li>
                    <li>Morales A, Korsakova E, Mansooralavi N, Soliman P, Jahanbani S, Olsen ML, et al. Evidence for
                        neuronal DNA damage in Rett Patient Brain. Disease Models & Mechanisms 2025:dmm.052358. <a
                                href="https://doi.org/10.1242/dmm.052358" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1242/dmm.052358</a>.
                    </li>
                    <li>Morgan R, Keeley D, Hazard ES, Allott EH, Wolf B, Savage SJ, et al. Fusion Genes in Prostate
                        Cancer: A Comparison in Men of African and European Descent. Biology 2022;11:625. <a
                                href="https://doi.org/10.3390/biology11050625" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.3390/biology11050625</a>.
                    </li>
                    <li>Mori T. Involvement of the p53-p16/RB pathway control mechanism in early-stage carcinogenesis in
                        head and neck squamous cell carcinoma. Pathology International 2022;72:577–88. <a
                                href="https://doi.org/10.1111/pin.13279" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1111/pin.13279</a>.
                    </li>
                    <li>Moser R, Gurley KE, Nikolova O, Qin G, Joshi R, Mendez E, et al. Synthetic lethal kinases in
                        Ras/p53 mutant squamous cell carcinoma. Oncogene 2022:1–15. <a
                                href="https://doi.org/10.1038/s41388-022-02330-w" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41388-022-02330-w</a>.
                    </li>
                    <li>Mukherjee K, Bhadaria H, Pillai S, Kumari M, Sarkar B. Nano-Informatics: Studies on Nano
                        Information Platforms and Their Application in Various Sectors. In: Sarkar B, Sonawane A,
                        editors. Biological Applications of Nanoparticles, Singapore: Springer Nature; 2023, p. 163–76.
                        <a href="https://doi.org/10.1007/978-981-99-3629-8_10" rel="noreferrer nofollow"
                           target="_blank">https://doi.org/10.1007/978-981-99-3629-8_10</a>.
                    </li>
                    <li>Musicant AM, Parag-Sharma K, Gong W, Sengupta M, Chatterjee A, Henry EC, et al. CRTC1/MAML2
                        directs a PGC-1α-IGF-1 circuit that confers vulnerability to PPARγ inhibition. Cell Reports
                        2021;34:108768. <a href="https://doi.org/10.1016/j.celrep.2021.108768" rel="noreferrer nofollow"
                                           target="_blank">https://doi.org/10.1016/j.celrep.2021.108768</a>.
                    </li>
                    <li>Nadikattu RR. The role of nanoinformatics in America 2023.</li>
                    <li>Naeem A, Suhail M, Basit A, Yali L, Xia ZM, Qin Z, et al. Chapter 19 - Convergence of artificial
                        intelligence and nanotechnology in the development of novel formulations for cancer treatment.
                        In: Philip A, Shahiwala A, Rashid M, Faiyazuddin Md, editors. A Handbook of Artificial
                        Intelligence in Drug Delivery, Academic Press; 2023, p. 499–529. <a
                                href="https://doi.org/10.1016/B978-0-323-89925-3.00019-8" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/B978-0-323-89925-3.00019-8</a>.
                    </li>
                    <li>Najafi A, Emami N, Samad-Soltani T. Integration of Genomics Data and Electronic Health Records
                        Toward Personalized Medicine: A Targeted Review. Frontiers in Health Informatics 2021;10:2021.
                        <a href="https://doi.org/10.30699/fhi.v10i1.299" rel="noreferrer nofollow" target="_blank">https://doi.org/10.30699/fhi.v10i1.299</a>.
                    </li>
                    <li>Nakata E, Ennishi D, Osone T, Ninomiya K, Tomida S, Itano T, et al. Real‐World Data of
                        Comprehensive Cancer Genomic Profiling Tests Performed in the Routine Clinical Setting in
                        Sarcoma. Cancer Medicine 2025;14. <a href="https://doi.org/10.1002/cam4.71098"
                                                             rel="noreferrer nofollow" target="_blank">https://doi.org/10.1002/cam4.71098</a>.
                    </li>
                    <li>Nakata E, Osone T, Ogawa T, Taguchi T, Hattori K, Kohsaka S. Prevalence of neurotrophic
                        tropomyosin receptor kinase ( NTRK ) fusion gene positivity in patients with solid tumors in
                        Japan. Cancer Medicine 2024;13:e7351. <a href="https://doi.org/10.1002/cam4.7351"
                                                                 rel="noreferrer nofollow" target="_blank">https://doi.org/10.1002/cam4.7351</a>.
                    </li>
                    <li>Nanopinion - ECHA. 2023. https://euon.echa.europa.eu/nanopinion/-/blogs/url (accessed November
                        30, 2023).
                    </li>
                    <li>Narasimhamurthy M, Le A, Boruah N, Moses R, Kelly G, Bleiweiss I, et al. Clinicopathologic
                        Features of Breast Tumors in Germline TP53 Variant–Associated Li-Fraumeni Syndrome. American
                        Journal of Surgical Pathology 2024. <a href="https://doi.org/10.1097/PAS.0000000000002338"
                                                               rel="noreferrer nofollow" target="_blank">https://doi.org/10.1097/PAS.0000000000002338</a>.
                    </li>
                    <li>Navale V, Bourne PE. Cloud computing applications for biomedical science: A perspective. PLoS
                        Computational Biology 2018;14. <a href="https://doi.org/10.1371/journal.pcbi.1006144"
                                                          rel="noreferrer nofollow" target="_blank">https://doi.org/10.1371/journal.pcbi.1006144</a>.
                    </li>
                    <li>Nguyen DT, Davidson T, Maclean F, Dorwal P, Lee YC, Qiu MR. Oligometastatic Renal Ewing Sarcoma
                        With FUS::ERG Fusion: A Case Report And Literature Review. Sage Open Pathology
                        2025;18:30502098251336541. <a href="https://doi.org/10.1177/30502098251336541"
                                                      rel="noreferrer nofollow" target="_blank">https://doi.org/10.1177/30502098251336541</a>.
                    </li>
                    <li>Nishio M, Paz-Ares L, Reck M, Nagakawa K, Garon EB, Popat S, et al. RELAY, Ramucirumab plus
                        Erlotinib (RAM+ERL) in Untreated Metastatic EGFR-Mutant NSCLC (EGFR+ NSCLC): Association between
                        TP53 Status and Clinical Outcome. Clinical Lung Cancer 2023. <a
                                href="https://doi.org/10.1016/j.cllc.2023.02.010" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.cllc.2023.02.010</a>.
                    </li>
                    <li>Njoya K, Zayed H, Sun L, Alson D, Aina O, Khan S, et al. Natural history of SPP1 signaling in
                        NF1 tumors. Npj Precis Onc 2025;9:320. <a href="https://doi.org/10.1038/s41698-025-01078-2"
                                                                  rel="noreferrer nofollow" target="_blank">https://doi.org/10.1038/s41698-025-01078-2</a>.
                    </li>
                    <li>Noller K, Botsis T, Camara PG, Ciotti L, Cooper LAd, Goecks J, et al. Informatics at the
                        Frontier of Cancer Research. Cancer Research 2025. <a
                                href="https://doi.org/10.1158/0008-5472.can-24-2829" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1158/0008-5472.can-24-2829</a>.
                    </li>
                    <li>Nyman J, Denize T, Bakouny Z, Labaki C, Titchen BM, Bi K, et al. Spatially aware deep learning
                        reveals tumor heterogeneity patterns that encode distinct kidney cancer states. Cancer Biology;
                        2023. <a href="https://doi.org/10.1101/2023.01.18.524545" rel="noreferrer nofollow"
                                 target="_blank">https://doi.org/10.1101/2023.01.18.524545</a>.
                    </li>
                    <li>Nzitakera A, Surwumwe JB, Ndoricyimpaye EL, Uwamungu S, Uwamariya D, Manirakiza F, et al. The
                        spectrum of TP53 mutations in Rwandan patients with gastric cancer. Genes and Environ 2024;46:8.
                        <a href="https://doi.org/10.1186/s41021-024-00302-y" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1186/s41021-024-00302-y</a>.
                    </li>
                    <li>Oak N, Cherniack AD, Mashl RJ, Carrot-Zhang J, Chambwe N, Damrauer JS, et al. Ancestry-specific
                        predisposing germline variants in cancer. Genome Medicine 2020;12. <a
                                href="https://doi.org/10.1186/s13073-020-00744-3" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1186/s13073-020-00744-3</a>.
                    </li>
                    <li>O’Grady N, Gibbs DL, Abdilleh K, Asare A, Asare S, Venters S, et al. PRoBE the cloud toolkit:
                        finding the best biomarkers of drug response within a breast cancer clinical trial. JAMIA Open
                        2021;4. <a href="https://doi.org/10.1093/jamiaopen/ooab038" rel="noreferrer nofollow"
                                   target="_blank">https://doi.org/10.1093/jamiaopen/ooab038</a>.
                    </li>
                    <li>Omole AO, Affonso De Oliveira JF, Sutorus L, Karan S, Zhao Z, Neun BW, et al. Cellular fate of a
                        plant virus immunotherapy candidate. Commun Biol 2024;7:1382. <a
                                href="https://doi.org/10.1038/s42003-024-06982-0" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s42003-024-06982-0</a>.
                    </li>
                    <li>Omran M. Heritable TP53-related cancer syndrome in Sweden : characterisation of
                        genotype-phenotype correlation and surveillance. Inst för onkologi-patologi / Dept of
                        Oncology-Pathology; 2023.
                    </li>
                    <li>Omran M, Liu Y, Sun Zhang A, Poluha A, Stenmark-Askmalm M, Persson F, et al. Characterisation of
                        heritable TP53-related cancer syndrome in Sweden—a nationwide study of genotype-phenotype
                        correlations in 90 families. Eur J Hum Genet 2025. <a
                                href="https://doi.org/10.1038/s41431-024-01753-1" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41431-024-01753-1</a>.
                    </li>
                    <li>Opdensteinen P, Affonso De Oliveira JF, Jung S, Steinmetz NF. Toward Translation of Cowpea
                        Mosaic Virus Intratumoral Immunotherapy With a Scalable Production Process. Plant Biotechnology
                        Journal 2025:pbi.70281. <a href="https://doi.org/10.1111/pbi.70281" rel="noreferrer nofollow"
                                                   target="_blank">https://doi.org/10.1111/pbi.70281</a>.
                    </li>
                    <li>Orsini A, Mastracci L, Bozzarelli I, Ferrari A, Isidori F, Fiocca R, et al. Correlations between
                        Molecular Alterations, Histopathological Characteristics, and Poor Prognosis in Esophageal
                        Adenocarcinoma. Cancers 2023;15:1408. <a href="https://doi.org/10.3390/cancers15051408"
                                                                 rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/cancers15051408</a>.
                    </li>
                    <li>Orzan F, De Bacco F, Lazzarini E, Crisafulli G, Gasparini A, Dipasquale A, et al. Liquid Biopsy
                        of Cerebrospinal Fluid Enables Selective Profiling of Glioma Molecular Subtypes at First
                        Clinical Presentation. Clin Cancer Res 2023;29:1252–66. <a
                                href="https://doi.org/10.1158/1078-0432.CCR-22-2903" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1158/1078-0432.CCR-22-2903</a>.
                    </li>
                    <li>Östlund D. Genomics in the Cloud. 2021.</li>
                    <li>Padella A, Simonetti G, Paciello G, Giotopoulos G, Baldazzi C, Righi S, et al. Novel and rare
                        fusion transcripts involving transcription factors and tumor suppressor genes in acute myeloid
                        leukemia. Cancers 2019;11. <a href="https://doi.org/10.3390/cancers11121951"
                                                      rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/cancers11121951</a>.
                    </li>
                    <li>Pagani F, Orzan F, Lago S, De Bacco F, Prelli M, Cominelli M, et al. Concurrent RB1 and P53
                        pathway disruption predisposes to the development of a primitive neuronal component in
                        high-grade gliomas depending on MYC-driven EBF3 transcription. Acta Neuropathol 2025;149:8. <a
                                href="https://doi.org/10.1007/s00401-025-02845-y" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1007/s00401-025-02845-y</a>.
                    </li>
                    <li>Pal S, Mondal S, Das G, Khatua S, Ghosh Z. Big data in biology: The hope and present-day
                        challenges in it. Gene Reports 2020;21:100869. <a
                                href="https://doi.org/10.1016/j.genrep.2020.100869" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.genrep.2020.100869</a>.
                    </li>
                    <li>Panagopoulos I, Andersen K, Brunetti M, Gorunova L, Davidson B, Lund-Iversen M, et al. Genetic
                        Pathways in Peritoneal Mesothelioma Tumorigenesis. Cancer Genomics & Proteomics 2023a;20:363–74.
                        <a href="https://doi.org/10.21873/cgp.20388" rel="noreferrer nofollow" target="_blank">https://doi.org/10.21873/cgp.20388</a>.
                    </li>
                    <li>PANAGOPOULOS I, ANDERSEN K, EILERT-OLSEN M, ZELLER B, MUNTHE-KAAS MC, BUECHNER J, et al.
                        Therapy-induced Deletion in 11q23 Leading to Fusion of KMT2A With ARHGEF12 and Development of B
                        Lineage Acute Lymphoplastic Leukemia in a Child Treated for Acute Myeloid Leukemia Caused by
                        t(9;11)(p21;q23)/ KMT2A-MLLT3. Cancer Genomics - Proteomics 2021;18:67–81. <a
                                href="https://doi.org/10.21873/cgp.20242" rel="noreferrer nofollow" target="_blank">https://doi.org/10.21873/cgp.20242</a>.
                    </li>
                    <li>Panagopoulos I, Andersen K, Gorunova L, Davidson B, Micci F, Heim S. A Novel Cryptic
                        t(2;3)(p21;q25) Translocation Fuses the WWTR1 and PRKCE Genes in Uterine Leiomyoma With 3q- as
                        the Sole Visible Chromosome Abnormality. CANCER GENOMICS 2022a:11.
                    </li>
                    <li>Panagopoulos I, Andersen K, Gorunova L, Lobmaier I. Fusion of Platelet Derived Growth Factor
                        Receptor Alpha ( PDGFRA ) With Ubiquitin Specific Peptidase 8 ( USP8 ) in a Calcified Chondroid
                        Mesenchymal Neoplasm Harboring t(4;15)(q12;q21) as a Sole Aberration. Cancer Genomics Proteomics
                        2024;21:252–9. <a href="https://doi.org/10.21873/cgp.20444" rel="noreferrer nofollow"
                                          target="_blank">https://doi.org/10.21873/cgp.20444</a>.
                    </li>
                    <li>Panagopoulos I, Andersen K, Johannsdottir IMR, Micci F, Heim S. Novel MYCBP::EHD2 and
                        RUNX1::ZNF780A Fusion Genes in T-cell Acute Lymphoblastic Leukemia. CANCER GENOMICS 2023b.
                    </li>
                    <li>Panagopoulos I, Andersen K, Wik HS, Tandsæther MR, Micci F, Heim S. Acute Undifferentiated
                        Leukemia With a Balanced t(5;10)(q35;p12) Resulting in Fusion of HNRNPH1 With MLLT10. Cancer
                        Genomics & Proteomics 2023c;20:354–62. <a href="https://doi.org/10.21873/cgp.20387"
                                                                  rel="noreferrer nofollow" target="_blank">https://doi.org/10.21873/cgp.20387</a>.
                    </li>
                    <li>Panagopoulos Ioannis, Gorunova L, Andersen K, Lobmaier I, Heim S. Several Fusion Genes
                        Identified in a Spermatic Cord Leiomyoma With Rearrangements of Chromosome Arms 3p and 21q.
                        Cancer Genomics & Proteomics 2021a;18:531–42. <a href="https://doi.org/10.21873/cgp.20278"
                                                                         rel="noreferrer nofollow" target="_blank">https://doi.org/10.21873/cgp.20278</a>.
                    </li>
                    <li>Panagopoulos I, Gorunova L, Andersen K, Lund-Iversen M, Hognestad HR, Lobmaier I, et al.
                        Chromosomal Translocation t(5;12)(p13;q14) Leading to Fusion of High-mobility Group AT-hook 2
                        Gene With Intergenic Sequences From Chromosome Sub-Band 5p13.2 in Benign Myoid Neoplasms of the
                        Breast: A Second Case. Cancer Genomics & Proteomics 2022b;19:445–55. <a
                                href="https://doi.org/10.21873/cgp.20331" rel="noreferrer nofollow" target="_blank">https://doi.org/10.21873/cgp.20331</a>.
                    </li>
                    <li>Panagopoulos Ioannis, Gorunova L, Andersen K, Lund-Iversen M, Tafjord S, Micci F, et al. Fusion
                        of the Paired Box 3 (PAX3) and Myocardin (MYOCD) Genes in Pediatric Rhabdomyosarcoma. CANCER
                        GENOMICS 2021b:12.
                    </li>
                    <li>Panagopoulos I, GORUNOVA L, Lobmaier I, ANDERSEN K, LUND-IVERSEN M, MICCI F, et al. Fusion of
                        the COL4A5 Gene With NR2F2-AS1 in a Hemangioma Carrying a t(X;15)(q22;q26) Chromosomal
                        Translocation. Cancer Genomics - Proteomics 2020;17:383–90. <a
                                href="https://doi.org/10.21873/cgp.20197" rel="noreferrer nofollow" target="_blank">https://doi.org/10.21873/cgp.20197</a>.
                    </li>
                    <li>Panagopoulos I, Heim S. Neoplasia-associated Chromosome Translocations Resulting in Gene
                        Truncation. CANCER GENOMICS 2022:26.
                    </li>
                    <li>Panagopoulos I, Heim S. Interstitial Deletions Generating Fusion Genes. Cancer Genomics &
                        Proteomics 2021;18:167–96. <a href="https://doi.org/10.21873/cgp.20251"
                                                      rel="noreferrer nofollow" target="_blank">https://doi.org/10.21873/cgp.20251</a>.
                    </li>
                    <li>Panagopoulos I, Andersen K, Eilert-Olsen M, Rognlien AG, Munthe-Kaas MC, Micci F, et al. Rare
                        KMT2A-ELL and Novel ZNF56-KMT2A Fusion Genes in Pediatric T-cell Acute Lymphoblastic Leukemia.
                        Cancer Genomics Proteomics 2021;18:121–31. <a href="https://doi.org/10.21873/cgp.20247"
                                                                      rel="noreferrer nofollow" target="_blank">https://doi.org/10.21873/cgp.20247</a>.
                    </li>
                    <li>Panayides AS, Amini A, Filipovic ND, Sharma A, Tsaftaris SA, Young A, et al. AI in Medical
                        Imaging Informatics: Current Challenges and Future Directions. IEEE Journal of Biomedical and
                        Health Informatics 2020;24:1837–57. <a href="https://doi.org/10.1109/JBHI.2020.2991043"
                                                               rel="noreferrer nofollow" target="_blank">https://doi.org/10.1109/JBHI.2020.2991043</a>.
                    </li>
                    <li>Panchalingam S, Kasivelu G, Jayaraman M. Computational modeling and molecular dynamic
                        simulations of gold nanoparticles. Gold Nanoparticles, Nanomaterials and Nanocomposites,
                        Elsevier; 2025, p. 869–93. <a href="https://doi.org/10.1016/B978-0-443-15897-1.00022-4"
                                                      rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/B978-0-443-15897-1.00022-4</a>.
                    </li>
                    <li>Pandey M, Shah SK, Gromiha MM. Databases and computational algorithms for identifying cancer
                        hotspot residues and mutations in proteins. Protein Mutations, WORLD SCIENTIFIC; 2024, p.
                        229–59. <a href="https://doi.org/10.1142/9789811293269_0011" rel="noreferrer nofollow"
                                   target="_blank">https://doi.org/10.1142/9789811293269_0011</a>.
                    </li>
                    <li>Pandya P, Jethva M, Rubin E, Birnbaum RY, Braiman A, Isakov N. PICOT binding to
                        chromatin-associated EED negatively regulates cyclin D2 expression by increasing H3K27me3 at the
                        CCND2 gene promoter. Cell Death and Disease 2019;10. <a
                                href="https://doi.org/10.1038/s41419-019-1935-0" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41419-019-1935-0</a>.
                    </li>
                    <li>Park H, Shapiro GI, Gao X, Mahipal A, Starr J, Furqan M, et al. Phase Ib study of eprenetapopt
                        (APR-246) in combination with pembrolizumab in patients with advanced or metastatic solid
                        tumors☆. ESMO Open 2022;7:100573. <a href="https://doi.org/10.1016/j.esmoop.2022.100573"
                                                             rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.esmoop.2022.100573</a>.
                    </li>
                    <li>Park J-C, Kim Y-J, Hwang G-H, Kang CY, Bae S, Cha H-J. Enhancing genome editing in hPSCs through
                        dual inhibition of DNA damage response and repair pathways. Nat Commun 2024;15:4002. <a
                                href="https://doi.org/10.1038/s41467-024-48111-9" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41467-024-48111-9</a>.
                    </li>
                    <li>Park SH, Brugiolo M, Akerman M, Das S, Urbanski L, Geier A, et al. Differential Functions of
                        Splicing Factors in Mammary Transformation and Breast Cancer Metastasis. Cell Reports
                        2019;29:2672-2688.e7. <a href="https://doi.org/10.1016/j.celrep.2019.10.110"
                                                 rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.celrep.2019.10.110</a>.
                    </li>
                    <li>Payton M, Belmontes B, Hanestad K, Moriguchi J, Chen K, McCarter JD, et al. Small-molecule
                        inhibition of kinesin KIF18A reveals a mitotic vulnerability enriched in chromosomally unstable
                        cancers. Nat Cancer 2023:1–19. <a href="https://doi.org/10.1038/s43018-023-00699-5"
                                                          rel="noreferrer nofollow" target="_blank">https://doi.org/10.1038/s43018-023-00699-5</a>.
                    </li>
                    <li>Peart CR, Williams C, Pophaly SD, Neely BA, Gulland FMD, Adams DJ, et al. Hi-C scaffolded short-
                        and long-read genome assemblies of the California sea lion are broadly consistent for syntenic
                        inference across 45 million years of evolution. Molecular Ecology Resources 2021;n/a. <a
                                href="https://doi.org/10.1111/1755-0998.13443" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1111/1755-0998.13443</a>.
                    </li>
                    <li>Penkert J, Strüwe FJ, Dutzmann CM, Doergeloh BB, Montellier E, Freycon C, et al.
                        Genotype–phenotype associations within the Li-Fraumeni spectrum: a report from the German
                        Registry. Journal of Hematology & Oncology 2022;15:107. <a
                                href="https://doi.org/10.1186/s13045-022-01332-1" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1186/s13045-022-01332-1</a>.
                    </li>
                    <li>Persson M, Andersson MK, Mitani Y, Brandwein-Weber MS, Frierson HF, Moskaluk C, et al.
                        Rearrangements, Expression, and Clinical Significance of MYB and MYBL1 in Adenoid Cystic
                        Carcinoma: A Multi-Institutional Study. Cancers 2022;14:3691. <a
                                href="https://doi.org/10.3390/cancers14153691" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.3390/cancers14153691</a>.
                    </li>
                    <li>Piccolo SR, Mecham A, Golightly NP, Johnson JL, Miller DB. The ability to classify patients
                        based on gene-expression data varies by algorithm and performance metric. PLOS Computational
                        Biology 2022;18:e1009926. <a href="https://doi.org/10.1371/journal.pcbi.1009926"
                                                     rel="noreferrer nofollow" target="_blank">https://doi.org/10.1371/journal.pcbi.1009926</a>.
                    </li>
                    <li>Pinto EM, Fridman C, Figueiredo BC, Salvador H, Teixeira MR, Pinto C, et al. Multiple TP53
                        p.R337H haplotypes and implications for tumor susceptibility. Human Genetics and Genomics
                        Advances 2024;5:100244. <a href="https://doi.org/10.1016/j.xhgg.2023.100244"
                                                   rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.xhgg.2023.100244</a>.
                    </li>
                    <li>Poore GD, Kopylova E, Zhu Q, Carpenter C, Fraraccio S, Wandro S, et al. Microbiome analyses of
                        blood and tissues suggest cancer diagnostic approach. Nature 2020. <a
                                href="https://doi.org/10.1038/s41586-020-2095-1" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41586-020-2095-1</a>.
                    </li>
                    <li>Pös O, Radvanszky J, Styk J, Pös Z, Buglyó G, Kajsik M, et al. Copy Number Variation: Methods
                        and Clinical Applications. Applied Sciences 2021;11:819. <a
                                href="https://doi.org/10.3390/app11020819" rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/app11020819</a>.
                    </li>
                    <li>Poscente M, Tolomeo D, Arshadi A, Agostini A, L’Abbate A, Solimando AG, et al. Aggressive
                        systemic mastocytosis with the co-occurrence of PRKG2::PDGFRB, KAT6A::NCOA2, and RXRA::NOTCH1
                        fusion transcripts and a heterozygous RUNX1 frameshift mutation. Cancer Genetics
                        2024;284–285:5–11. <a href="https://doi.org/10.1016/j.cancergen.2024.03.002"
                                              rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.cancergen.2024.03.002</a>.
                    </li>
                    <li>Post AR, Ho N, Rasmussen E, Post I, Cho A, Hofer J, et al. Hypermedia-based software
                        architecture enables Test-Driven Development. JAMIA Open 2023;6:ooad089. <a
                                href="https://doi.org/10.1093/jamiaopen/ooad089" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1093/jamiaopen/ooad089</a>.
                    </li>
                    <li>Prasad R, Ghosh A, Patel V, Peng B, Mendes BB, Win EHA, et al. Voices of Nanomedicine: Blueprint
                        Guidelines for Collaboration in Addressing Global Unmet Medical Needs. ACS Nano 2025;19:2979–91.
                        <a href="https://doi.org/10.1021/acsnano.4c13513" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1021/acsnano.4c13513</a>.
                    </li>
                    <li>Privitera AP, Barresi V, Condorelli DF. Aberrations of Chromosomes 1 and 16 in Breast Cancer: A
                        Framework for Cooperation of Transcriptionally Dysregulated Genes. Cancers 2021;13:1585. <a
                                href="https://doi.org/10.3390/cancers13071585" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.3390/cancers13071585</a>.
                    </li>
                    <li>Pruessmann W, Kirfel J, Sailer V, Rose C. Dermatomyositis‐like skin eruptions under hydroxyurea
                        therapy conceal TP53 ‐mutated atypical keratinocytes: A histopathologic and molecular pathologic
                        case series. J Cutan Pathol 2024:cup.14690. <a href="https://doi.org/10.1111/cup.14690"
                                                                       rel="noreferrer nofollow" target="_blank">https://doi.org/10.1111/cup.14690</a>.
                    </li>
                    <li>Puls F, Agaimy A, Flucke U, Mentzel T, Sumathi VP, Ploegmakers M, et al. Recurrent fusions
                        between YAP1 and KMT2A in morphologically distinct neoplasms within the spectrum of low-grade
                        fibromyxoid sarcoma and sclerosing epithelioid fibrosarcoma. American Journal of Surgical
                        Pathology 2020;44:594–606. <a href="https://doi.org/10.1097/PAS.0000000000001423"
                                                      rel="noreferrer nofollow" target="_blank">https://doi.org/10.1097/PAS.0000000000001423</a>.
                    </li>
                    <li>Raghav P, Mann Z, Pandey P, Mohanty S. Systems Biology Resources and Their Applications to
                        Understand the Cancer, 2021, p. 1–35. <a href="https://doi.org/10.1007/978-981-15-4501-6_140-1"
                                                                 rel="noreferrer nofollow" target="_blank">https://doi.org/10.1007/978-981-15-4501-6_140-1</a>.
                    </li>
                    <li>Rai S, Singh MP, Sinha A, Srivastava A, Datta D, Srivastava S. Unravelling a novel CTNND1-RAB6A
                        fusion transcript: Implications in colon cancer cell migration. International Journal of
                        Biological Macromolecules 2024;262:129981. <a
                                href="https://doi.org/10.1016/j.ijbiomac.2024.129981" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.ijbiomac.2024.129981</a>.
                    </li>
                    <li>Rainville V, Lanic M-D, Drieux F, Piton N, Rizzo D, Jardin F, et al. Detection of
                        Cancer-Associated Gene Fusion by Ligation-Dependent RT-PCR. In: Delpy L, editor. RNA Analysis,
                        vol. 2962, New York, NY: Springer US; 2025, p. 25–37. <a
                                href="https://doi.org/10.1007/978-1-0716-4726-4_3" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1007/978-1-0716-4726-4_3</a>.
                    </li>
                    <li>Rajanala SH, Ghale R, Nandakumar S, Chadalavada K, Lee G-SM, Stopsack KH, et al. Quantifying Y
                        chromosome loss in primary and metastatic prostate cancer by chromosome painting. PLoS ONE
                        2024;19:e0301989. <a href="https://doi.org/10.1371/journal.pone.0301989"
                                             rel="noreferrer nofollow" target="_blank">https://doi.org/10.1371/journal.pone.0301989</a>.
                    </li>
                    <li>Ran R, Li L, Shi Z, Liu G, Jiang H, Fang L, et al. Disruption of tp53 leads to cutaneous nevus
                        and melanoma formation in Xenopus tropicalis. Molecular Oncology 2022;n/a. <a
                                href="https://doi.org/10.1002/1878-0261.13301" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1002/1878-0261.13301</a>.
                    </li>
                    <li>Ratajczak B, Przybyłowicz-Chalecka A, Czerwińska-Rybak J, Kanduła Z, Ustaszewski A, Gil L, et
                        al. The presence of additional cytogenetic aberrations in chronic myeloid leukemia cells at the
                        time of diagnosis or their appearance on tyrosine kinase inhibitor therapy predicts the imatinib
                        treatment failure. Leukemia Research 2023;132:107349. <a
                                href="https://doi.org/10.1016/j.leukres.2023.107349" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.leukres.2023.107349</a>.
                    </li>
                    <li>Reid JC, Golubeva D, Boyd AL, Hollands CG, Henly C, Orlando L, et al. Human pluripotent stem
                        cells identify molecular targets of trisomy 12 in chronic lymphocytic leukemia patients. Cell
                        Reports 2021;34. <a href="https://doi.org/10.1016/j.celrep.2021.108845"
                                            rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.celrep.2021.108845</a>.
                    </li>
                    <li>Ren H, Zhuang X, Rabczuk T. Dispersion relation computation in periodic media via nonlocal
                        operator methods. Nano Micro Mechanics Rev 2025;01:93–101. <a
                                href="https://doi.org/10.1142/S3082805825300021" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1142/S3082805825300021</a>.
                    </li>
                    <li>Romanovsky E, Kluck K, Ourailidis I, Menzel M, Beck S, Ball M, et al. Homogenous
                        TP53mut-associated tumor biology across mutation and cancer types revealed by transcriptome
                        analysis. Cell Death Discov 2023;9:1–14. <a href="https://doi.org/10.1038/s41420-023-01413-1"
                                                                    rel="noreferrer nofollow" target="_blank">https://doi.org/10.1038/s41420-023-01413-1</a>.
                    </li>
                    <li>Romero-Morelos P, González-Yebra AL, Muñoz-López D, Lara-Lona E, González-Yebra B. Frequencies
                        of BCR::ABL1 Transcripts in Patients with Chronic Myeloid Leukemia: A Meta-Analysis. Genes
                        2024;15:232. <a href="https://doi.org/10.3390/genes15020232" rel="noreferrer nofollow"
                                        target="_blank">https://doi.org/10.3390/genes15020232</a>.
                    </li>
                    <li>Rybacki K, Xu F, Deutsch HM, Ahsan MU, Chan J, Liang Z, et al. Combining panel-based and
                        whole-transcriptome-based gene fusion detection by long-read sequencing. Cell Reports Methods
                        2025:101111. <a href="https://doi.org/10.1016/j.crmeth.2025.101111" rel="noreferrer nofollow"
                                        target="_blank">https://doi.org/10.1016/j.crmeth.2025.101111</a>.
                    </li>
                    <li>Saba K. Genomic and Transcriptomic Analyses of Osteogenic Tumours of Bone. PhD Thesis. Lund
                        University, 2020.
                    </li>
                    <li>Sachwani K, Pasha R, Kaleem B, Mansoor N. Cytogenetic finding of simultaneous inversion of
                        chromosome 14 and tetrasomy 8 in a Pediatric B-Lymphoblastic Leukemia-a case report. Mol
                        Cytogenet 2025;18:15. <a href="https://doi.org/10.1186/s13039-025-00715-6"
                                                 rel="noreferrer nofollow" target="_blank">https://doi.org/10.1186/s13039-025-00715-6</a>.
                    </li>
                    <li>Sadoq B-E, Britel M, Bouajaj A, Maâlej R, Touhami A, Abid M, et al. A Review on Antibacterial
                        Activity of Nanoparticles. Biointerface Research in Applied Chemistry 2023. <a
                                href="https://doi.org/1.33263/BRIAC135.405" rel="noreferrer nofollow" target="_blank">https://doi.org/1.33263/BRIAC135.405</a>.
                    </li>
                    <li>Sahin Y, Wang YL, Pei J, Mansoor N, Styler M, Testa JR, et al. Multiple Genomic Alterations,
                        Including a Novel AFF4::IRF1 Fusion Gene, in a Treatment-Refractory Blastic Plasmacytoid
                        Dendritic-Cell Neoplasm: A Case Report and Literature Review. International Journal of Molecular
                        Sciences 2024;25:305. <a href="https://doi.org/10.3390/ijms25010305" rel="noreferrer nofollow"
                                                 target="_blank">https://doi.org/10.3390/ijms25010305</a>.
                    </li>
                    <li>Sahnane N, Rivera D, Libera L, Carnevali I, Banelli B, Facchi S, et al. Pyrosequencing Assay for
                        BRCA1 Methylation Analysis: Results from a Cross-Validation Study. The Journal of Molecular
                        Diagnostics 2023. <a href="https://doi.org/10.1016/j.jmoldx.2023.01.003"
                                             rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.jmoldx.2023.01.003</a>.
                    </li>
                    <li>Saliba J, Church AJ, Rao S, Danos A, Furtado LV, Laetsch T, et al. Standardized evidence-based
                        approach for assessment of oncogenic and clinical significance of NTRK fusions. Cancer Genetics
                        2022;264–265:50–9. <a href="https://doi.org/10.1016/j.cancergen.2022.03.001"
                                              rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.cancergen.2022.03.001</a>.
                    </li>
                    <li>Saltz J, Gupta R, Hou L, Kurc T, Singh P, Nguyen V, et al. Spatial Organization and Molecular
                        Correlation of Tumor-Infiltrating Lymphocytes Using Deep Learning on Pathology Images. Cell
                        Reports 2018;23:181-193.e7. <a href="https://doi.org/10.1016/j.celrep.2018.03.086"
                                                       rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.celrep.2018.03.086</a>.
                    </li>
                    <li>Samadi Avansar S, Nouri S, Hoseinzade M, Mortazavi R, Abdollahzadeh Sangrody M, Nemati Manshour
                        K. Application of Machine Learning Models in Forecasting Prolonged Length of Stay After Cancer
                        Surgery Using Electronic Health Records. InfoSci Trend 2025;2:80–9. <a
                                href="https://doi.org/10.61186/ist.202502.05.08" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.61186/ist.202502.05.08</a>.
                    </li>
                    <li>Sasaki K, Takahashi S, Ouchi K, Otsuki Y, Wakayama S, Ishioka C. Different impacts of TP53
                        mutations on cell cycle-related gene expression among cancer types. Sci Rep 2023;13:4868. <a
                                href="https://doi.org/10.1038/s41598-023-32092-8" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41598-023-32092-8</a>.
                    </li>
                    <li>Sassi H, Meddeb R, Cherif MA, Nasr C, Riahi A, Hannachi S, et al. Li–Fraumeni syndrome in
                        Tunisian carriers with different and rare tumor phenotype: genotype–phenotype correlation. BMC
                        Med Genomics 2022;15:44. <a href="https://doi.org/10.1186/s12920-022-01189-w"
                                                    rel="noreferrer nofollow" target="_blank">https://doi.org/10.1186/s12920-022-01189-w</a>.
                    </li>
                    <li>Schimmelpfennig C, Rade M, Füssel S, Löffler D, Blumert C, Bertram C, et al. Characterization
                        and evaluation of gene fusions as a measure of genetic instability and disease prognosis in
                        prostate cancer. BMC Cancer 2023;23:575. <a href="https://doi.org/10.1186/s12885-023-11019-6"
                                                                    rel="noreferrer nofollow" target="_blank">https://doi.org/10.1186/s12885-023-11019-6</a>.
                    </li>
                    <li>Schulz WA. Cancer Genetics. In: Schulz WA, editor. Molecular Biology of Human Cancers, Cham:
                        Springer International Publishing; 2023, p. 29–49. <a
                                href="https://doi.org/10.1007/978-3-031-16286-2_2" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1007/978-3-031-16286-2_2</a>.
                    </li>
                    <li>Schwede M, Waldron L, Mok SC, Wei W, Basunia A, Merritt MA, et al. The impact of stroma on the
                        discovery of molecular subtypes and prognostic gene signatures in serous ovarian cancer. vol.
                        202. bioRxiv; 2018. <a href="https://doi.org/10.1101/496406" rel="noreferrer nofollow"
                                               target="_blank">https://doi.org/10.1101/496406</a>.
                    </li>
                    <li>Sebastian AM, Peter D. Artificial Intelligence in Cancer Research: Trends, Challenges and Future
                        Directions. Life 2022;12:1991. <a href="https://doi.org/10.3390/life12121991"
                                                          rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/life12121991</a>.
                    </li>
                    <li>Sekiguchi K, Motoo I, Ando T, Noguchi A, Fukuda R, Nomura K, et al. Early-onset Colorectal
                        Cancer in a Patient with Li-Fraumeni Syndrome: A Case Series and Literature Review. Intern Med
                        2025:4945–24. <a href="https://doi.org/10.2169/internalmedicine.4945-24"
                                         rel="noreferrer nofollow" target="_blank">https://doi.org/10.2169/internalmedicine.4945-24</a>.
                    </li>
                    <li>Settino M, Cannataro M. Survey of main tools for querying and analyzing TCGA Data. Proceedings -
                        2018 IEEE International Conference on Bioinformatics and Biomedicine, BIBM 2018, Institute of
                        Electrical and Electronics Engineers Inc.; 2019, p. 1711–8. <a
                                href="https://doi.org/10.1109/BIBM.2018.8621270" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1109/BIBM.2018.8621270</a>.
                    </li>
                    <li>Sharma A, Anjum Z, Raza K, Sharma N, Kurmi BD. Nanoinformatics: Emerging technology for
                        prediction and controlling of biological performance of nanomedicines. Computational Toxicology
                        2025;35:100378. <a href="https://doi.org/10.1016/j.comtox.2025.100378" rel="noreferrer nofollow"
                                           target="_blank">https://doi.org/10.1016/j.comtox.2025.100378</a>.
                    </li>
                    <li>Sharma R, Kumar AVS, Kumar K, editors. Computational biology in drug discovery and repurposing.
                        First edition. Palm Bay, FL : Boca Raton, FL: Apple Academic Press ; CRC Press; 2025.
                    </li>
                    <li>Shen J, Lu L, Chen Z, Guo W, Wang S, Liu Z, et al. Multi-omics analysis constructs a novel
                        neuroendocrine prostate cancer classifier and classification system. Sci Rep 2025;15:13901. <a
                                href="https://doi.org/10.1038/s41598-025-96683-3" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41598-025-96683-3</a>.
                    </li>
                    <li>Sherif ZA, Ogunwobi OO, Ressom HW. Mechanisms and technologies in cancer epigenetics. Front
                        Oncol 2025;14:1513654. <a href="https://doi.org/10.3389/fonc.2024.1513654"
                                                  rel="noreferrer nofollow" target="_blank">https://doi.org/10.3389/fonc.2024.1513654</a>.
                    </li>
                    <li>Shetty D, Amare PK, Mohanty P, Talker E, Chaubal K, Jain Hemani, et al. Investigating the
                        clinical, hematological and cytogenetic profile of endoreduplicated hypodiploids in BCP-ALL.
                        Blood Cells, Molecules, and Diseases 2020;85:102465. <a
                                href="https://doi.org/10.1016/j.bcmd.2020.102465" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.bcmd.2020.102465</a>.
                    </li>
                    <li>Shetty D, Mohanty P, Talker E, Jain H, Chaubal K, Tembhare P, et al. Importance of conventional
                        cytogenetics in the identification of ins(19;X)(q13.1;p11.2q28) and t(1;11)(q10;p10), both,
                        novel cytogenetic abnormalities in a pediatric AML case. Cancer Genetics 2021;256–257:17–20. <a
                                href="https://doi.org/10.1016/j.cancergen.2021.03.002" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.cancergen.2021.03.002</a>.
                    </li>
                    <li>Shinomiya N, Shimada Y, editors. Applications of Cell Culture. 1st ed. 2025. Singapore: Springer
                        Nature Singapore; 2025. <a href="https://doi.org/10.1007/978-981-95-0587-6"
                                                   rel="noreferrer nofollow" target="_blank">https://doi.org/10.1007/978-981-95-0587-6</a>.
                    </li>
                    <li>Simms A, Minasov N, Steinmetz NF. Storage stability of cowpea mosaic virus cancer drug candidate
                        – a two-year update. Nanomedicine 2025:1–7. <a
                                href="https://doi.org/10.1080/17435889.2025.2544522" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1080/17435889.2025.2544522</a>.
                    </li>
                    <li>Singh A, Malhotra L, Mishra A, Kundral S, Tiwari PK, Kumar S, et al. The R337C mutation in the
                        p53 oligomerization domain affects the regulatory domain and its ability to bind response
                        elements: Evidence based on structural and biophysical studies. Archives of Biochemistry and
                        Biophysics 2025;768:110381. <a href="https://doi.org/10.1016/j.abb.2025.110381"
                                                       rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.abb.2025.110381</a>.
                    </li>
                    <li>Singh M, Sharma P, Bhatia P, Trehan A, Thakur R, Sreedharanunni S. Integrated analysis of
                        transcriptome and genome variations in pediatric T cell acute lymphoblastic leukemia: data from
                        north Indian tertiary care center. BMC Cancer 2024;24:325. <a
                                href="https://doi.org/10.1186/s12885-024-12063-6" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1186/s12885-024-12063-6</a>.
                    </li>
                    <li>Sinoliya P, Niraj RRK, Sharma V. Cracking the Code: How Nano-Informatics is Crafting Intelligent
                        Nano-Weapons to Outsmart Multiple Drug Resistance (MDR). PNT 2025;13. <a
                                href="https://doi.org/10.2174/0122117385383102250818052847" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.2174/0122117385383102250818052847</a>.
                    </li>
                    <li>Sobahy TM, Tashkandi G, Bahussain D, Al-Harbi R. Clinically actionable cancer somatic variants
                        (CACSV): a tumor interpreted dataset for analytical workflows. BMC Medical Genomics 2022;15:95.
                        <a href="https://doi.org/10.1186/s12920-022-01235-7" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1186/s12920-022-01235-7</a>.
                    </li>
                    <li>Sohail M, Ikram P. Cancer Genomics and IT Infrastructure for data storage and analysis. Journal
                        of Carcinogenesis 2022;21.
                    </li>
                    <li>Solmaz M, Lane A, Gonen B, Akmamedova O, Gunes MH, Komurov K, et al. Graphical data mining of
                        cancer mechanisms with SEMA. Bioinformatics 2019;35:4413–8. <a
                                href="https://doi.org/10.1093/bioinformatics/btz303" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1093/bioinformatics/btz303</a>.
                    </li>
                    <li>Son HJ, Lee JH. Novel Four-Way Variant Translocation, t(1;9;22;16)(q21;q34;q11.2;q24), in a
                        Patient with Chronic Myeloid Leukemia. Diagnostics 2024;14:303. <a
                                href="https://doi.org/10.3390/diagnostics14030303" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.3390/diagnostics14030303</a>.
                    </li>
                    <li>Soriani S, Guido V, Bertani G, Cesana C, Motta V, De Canal G, et al. BCR/ABL1 fluorescence in
                        situ hybridization fusion signals on both copies of chromosome 22 in a Philadelphia-masked
                        chronic myeloid leukemia case: implication for the therapy. Hematol Rep 2021;13. <a
                                href="https://doi.org/10.4081/hr.2021.8795" rel="noreferrer nofollow" target="_blank">https://doi.org/10.4081/hr.2021.8795</a>.
                    </li>
                    <li>Sorokin M, Rabushko E, Rozenberg JM, Mohammad T, Seryakov A, Sekacheva M, et al. Clinically
                        relevant fusion oncogenes: detection and practical implications. Ther Adv Med Oncol
                        2022;14:17588359221144108. <a href="https://doi.org/10.1177/17588359221144108"
                                                      rel="noreferrer nofollow" target="_blank">https://doi.org/10.1177/17588359221144108</a>.
                    </li>
                    <li>Soukup J, Valtr O, Brtkova J, Zoul Z, Staniczkova-Zambo I, Hojny J, et al. Soft tissue sarcoma
                        with ZC3H7B::BCOR fusion in a male mimicking low-grade fibromyxoid sarcoma – A case report.
                        Pathology - Research and Practice 2023;251:154831. <a
                                href="https://doi.org/10.1016/j.prp.2023.154831" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.prp.2023.154831</a>.
                    </li>
                    <li>Steffens Reinhardt L, Groen K, Xavier A, Avery-Kiejda KA. p53 Dysregulation in Breast Cancer:
                        Insights on Mutations in the TP53 Network and p53 Isoform Expression. International Journal of
                        Molecular Sciences 2023;24:10078. <a href="https://doi.org/10.3390/ijms241210078"
                                                             rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/ijms241210078</a>.
                    </li>
                    <li>Steinbuck MP, Cabana-Puig X, Palmer E, Jung MM, Williams T, Osaer K, et al. Lymph Node Targeted
                        AMP-peptide Vaccines Generate Functional T cell Immunity Against Mutant p53 and BRAF 2023.
                    </li>
                    <li>Stembalska A, Pesz K. The role of genetic counselling in oncology. Nowotwory Journal of Oncology
                        2022;72:207–10. <a href="https://doi.org/10.5603/NJO.2022.0030" rel="noreferrer nofollow"
                                           target="_blank">https://doi.org/10.5603/NJO.2022.0030</a>.
                    </li>
                    <li>Stengel A, Shahswar R, Haferlach T, Walter W, Hutter S, Meggendorfer M, et al. Whole
                        transcriptome sequencing detects a large number of novel fusion transcripts in patients with AML
                        and MDS. Blood Advances 2020;4:5393–401. <a
                                href="https://doi.org/10.1182/bloodadvances.2020003007" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1182/bloodadvances.2020003007</a>.
                    </li>
                    <li>Stenman G, Fehr A, Skálová A, Vander Poorten V, Hellquist H, Mikkelsen LH, et al. Chromosome
                        Translocations, Gene Fusions, and Their Molecular Consequences in Pleomorphic Salivary Gland
                        Adenomas. Biomedicines 2022;10:1970. <a href="https://doi.org/10.3390/biomedicines10081970"
                                                                rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/biomedicines10081970</a>.
                    </li>
                    <li>Striker SS, Wilferd SF, Lewis EM, O’Connor SA, Plaisier CL. Systematic integration of
                        protein-affecting mutations, gene fusions, and copy number alterations into a comprehensive
                        somatic mutational profile. Cell Reports Methods 2023;3:100442. <a
                                href="https://doi.org/10.1016/j.crmeth.2023.100442" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.crmeth.2023.100442</a>.
                    </li>
                    <li>Stružinská I, Hájková N, Hojný J, Krkavcová E, Michálková R, Bui QH, et al. Somatic Genomic and
                        Transcriptomic Characterization of Primary Ovarian Serous Borderline Tumors and Low-Grade Serous
                        Carcinomas. The Journal of Molecular Diagnostics 2024;26:257–66. <a
                                href="https://doi.org/10.1016/j.jmoldx.2023.12.004" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.jmoldx.2023.12.004</a>.
                    </li>
                    <li>Stružinská I, Hájková N, Hojný J, Krkavcová E, Michálková R, Dvořák J, et al. A comprehensive
                        molecular analysis of 113 primary ovarian clear cell carcinomas reveals common therapeutically
                        significant aberrations. Diagnostic Pathology 2023;18:72. <a
                                href="https://doi.org/10.1186/s13000-023-01358-0" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1186/s13000-023-01358-0</a>.
                    </li>
                    <li>Suchan M, Wuerdemann N, Wagner S, Langer C, Arens C, Johannsen J, et al. Histological and
                        genetic criteria define a clinically relevant subgroup of HPV-positive oropharyngeal carcinoma.
                        Oral Oncology 2025;162:107209. <a href="https://doi.org/10.1016/j.oraloncology.2025.107209"
                                                          rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.oraloncology.2025.107209</a>.
                    </li>
                    <li>Swiatkowska A. p53 and Its Isoforms in Renal Cell Carcinoma—Do They Matter? Biomedicines
                        2022;10:1330. <a href="https://doi.org/10.3390/biomedicines10061330" rel="noreferrer nofollow"
                                         target="_blank">https://doi.org/10.3390/biomedicines10061330</a>.
                    </li>
                    <li>Szallasi Z. An introduction to the computational challenges in next generation sequencing.
                        Communications in Computer and Information Science, vol. 822, Springer Verlag; 2018, p. 37–45.
                        <a href="https://doi.org/10.1007/978-3-319-96553-6_3" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1007/978-3-319-96553-6_3</a>.
                    </li>
                    <li>Talebi A, Rokni P, Kerachian MA. Transcriptome analysis of colorectal cancer liver metastasis:
                        The importance of long non-coding RNAs and fusion transcripts in the disease pathogenesis.
                        Molecular and Cellular Probes 2022a;63:101816. <a
                                href="https://doi.org/10.1016/j.mcp.2022.101816" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.mcp.2022.101816</a>.
                    </li>
                    <li>Talebi A, Shahidsales S, Aliakbarian M, Pezeshki Rad M, Kerachian MA. Oncogenic fusion
                        transcript analysis identified ADAP1-NOC4L, potentially associated with metastatic colorectal
                        cancer. Cancer Medicine 2022b;n/a. <a href="https://doi.org/10.1002/cam4.4943"
                                                              rel="noreferrer nofollow" target="_blank">https://doi.org/10.1002/cam4.4943</a>.
                    </li>
                    <li>Talka M. Optimized sampling protocol for fusion gene detection in acute leukemia diagnostics
                        using RNA sequencing 2022.
                    </li>
                    <li>Tam B, Lagniton PNP, Da Luz M, Zhao B, Sinha S, Lei CL, et al. Comprehensive classification of
                        TP53 somatic missense variants based on their impact on p53 structural stability. Briefings in
                        Bioinformatics 2024;25:bbae400. <a href="https://doi.org/10.1093/bib/bbae400"
                                                           rel="noreferrer nofollow" target="_blank">https://doi.org/10.1093/bib/bbae400</a>.
                    </li>
                    <li>Tam B, Qin Z, Zhao B, Wang SM, Lei CL. Integration of deep learning with Ramachandran plot
                        molecular dynamics simulation for genetic variant classification. iScience 2023;26:106122. <a
                                href="https://doi.org/10.1016/j.isci.2023.106122" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.isci.2023.106122</a>.
                    </li>
                    <li>Tang L, Hill MC, He M, Chen J, Wang Z, Ellinor PT, et al. A 3D Genome Atlas of Genetic Variants
                        and Their Pathological Effects in Cancer. Advanced Science 2025:2408420. <a
                                href="https://doi.org/10.1002/advs.202408420" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1002/advs.202408420</a>.
                    </li>
                    <li>Taniue K, Akimitsu N. Fusion Genes and RNAs in Cancer Development. Non-coding RNA 2021, 7, 10
                        2021.
                    </li>
                    <li>Taniue Kenzui, Akimitsu N. non-coding RNA Fusion Genes and RNAs in Cancer Development 2021. <a
                            href="https://doi.org/10.3390/ncrna7010010" rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/ncrna7010010</a>.
                    </li>
                    <li>Tarraubella LC. Computational Infrastructures for biomolecular research Laia Codó Tarraubella
                        Aquesta tesi doctoral està subjecta a la llicència Reconeixement 4.0. Espanya de Creative
                        Commons. Esta tesis doctoral está sujeta a la licencia Reconocimiento 4.0. España de Cr. 2019.
                    </li>
                    <li>Tatlow PJ, Piccolo SR. A cloud-based workflow to quantify transcript-expression levels in public
                        cancer compendia. Scientific Reports 2016;6:39259. <a href="https://doi.org/10.1038/srep39259"
                                                                              rel="noreferrer nofollow" target="_blank">https://doi.org/10.1038/srep39259</a>.
                    </li>
                    <li>Teku G, Nilsson J, Magnusson L, Sydow S, Flucke U, Puls F, et al. Insertion of the CXXC domain
                        of KMT2A into YAP1: An unusual mechanism behind the formation of a chimeric oncogenic protein.
                        Genes, Chromosomes and Cancer 2023;n/a. <a href="https://doi.org/10.1002/gcc.23176"
                                                                   rel="noreferrer nofollow" target="_blank">https://doi.org/10.1002/gcc.23176</a>.
                    </li>
                    <li>Teramura Y, Tanaka M, Yamazaki Y, Yamashita K, Takazawa Y, Ae K, et al. Identification of Novel
                        Fusion Genes in Bone and Soft Tissue Sarcoma and Their Implication in the Generation of a Mouse
                        Model. Cancers 2020;12:2345. <a href="https://doi.org/10.3390/cancers12092345"
                                                        rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/cancers12092345</a>.
                    </li>
                    <li>Thanedar S, Heng E, Ju D, Zhang K, Heng HH. Studying the Dynamics of Tunneling Tubes and
                        Cellular Spheres. In: Ye JC, Heng HH, editors. Cancer Cytogenetics and Cytogenomics, vol. 2825,
                        New York, NY: Springer US; 2024, p. 333–43. <a
                                href="https://doi.org/10.1007/978-1-0716-3946-7_19" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1007/978-1-0716-3946-7_19</a>.
                    </li>
                    <li>Thangudu RR, Holck M, Singhal D, Pilozzi A, Edwards N, Rudnick PA, et al. NCI’s Proteomic Data
                        Commons: A Cloud-Based Proteomics Repository Empowering Comprehensive Cancer Analysis Through
                        Cross-Referencing with Genomic and Imaging Data. Cancer Research Communications 2024. <a
                                href="https://doi.org/10.1158/2767-9764.CRC-24-0243" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1158/2767-9764.CRC-24-0243</a>.
                    </li>
                    <li>The Nordic Society of Paediatric Haematology and Oncology (NOPHO) ALL2008 trial. 2023.</li>
                    <li>Thomson R. How to run statistics inside BigQuery. Medium 2023.
                        https://medium.com/@jrossthomson/how-to-run-statistics-inside-bigquery-95c0c6864f23 (accessed
                        June 5, 2023).
                    </li>
                    <li>Thorsson VV, Gibbs DL, Brown SD, Wolf D, Bortone DS, Ou Yang T-HH, et al. The Immune Landscape
                        of Cancer. Immunity 2018;48:812-830.e14. <a href="https://doi.org/10.1016/j.immuni.2018.03.023"
                                                                    rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.immuni.2018.03.023</a>.
                    </li>
                    <li>Toh C. Chromosomal scale length variations as a genetic risk score for predicting complex human
                        diseases in large scale genomic datasets. UC Irvine, 2021.
                    </li>
                    <li>Toh C, Brody JP. Genetic risk score for ovarian cancer based on chromosomal-scale length
                        variation. BioData Mining 2021;14:18. <a href="https://doi.org/10.1186/s13040-021-00253-y"
                                                                 rel="noreferrer nofollow" target="_blank">https://doi.org/10.1186/s13040-021-00253-y</a>.
                    </li>
                    <li>Toh C, Brody JP. Chromosomal scale length variation of germline DNA can predict individual
                        cancer risk. 2020. <a href="https://doi.org/10.1101/303339" rel="noreferrer nofollow"
                                              target="_blank">https://doi.org/10.1101/303339</a>.
                    </li>
                    <li>Tolomeo D, Agostini A, Solimando AG, Cunsolo CL, Cimarosto L, Palumbo O, et al. A
                        t(4;13)(q21;q14) translocation in B-cell chronic lymphocytic leukemia causing concomitant
                        homozygous DLEU2/miR15a/miR16-1 and heterozygous ARHGAP24 deletions. Cancer Genetics
                        2023;272–273:16–22. <a href="https://doi.org/10.1016/j.cancergen.2023.01.003"
                                               rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.cancergen.2023.01.003</a>.
                    </li>
                    <li>Torcivia JP, Mazumder R. Scanning window analysis of non-coding regions within normal-tumor
                        whole-genome sequence samples. Briefings in Bioinformatics 2021;22. <a
                                href="https://doi.org/10.1093/bib/bbaa203" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1093/bib/bbaa203</a>.
                    </li>
                    <li>Torrezan GT, Mehdipour P. Women in cancer genetics vol II: 2022. Frontiers Media SA; 2023.</li>
                    <li>Trac QT. Statistical and computational methodologies for omics data analyses and drug response
                        prediction. thesis. Karolinska Institutet, 2024. <a href="https://doi.org/10.69622/26053291.v1"
                                                                            rel="noreferrer nofollow" target="_blank">https://doi.org/10.69622/26053291.v1</a>.
                    </li>
                    <li>Trac QT, Zhou T, Pawitan Y, Vu TN. Discovery of druggable cancer-specific pathways with
                        application in acute myeloid leukemia. GigaScience 2022;11:giac091. <a
                                href="https://doi.org/10.1093/gigascience/giac091" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1093/gigascience/giac091</a>.
                    </li>
                    <li>Tsang H, Addepalli K. National Cancer Institute Cloud Resources. dl.acm.org, 2018, p. 707–707.
                        <a href="https://doi.org/10.1145/3233547.3233671" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1145/3233547.3233671</a>.
                    </li>
                    <li>Tucker J, Bonema S, García-Varela R, Denu R, Hu Y, McGregor S, et al. Misaligned Chromosomes are
                        a Major Source of Chromosomal Instability in Breast Cancer. Cancer Research Communications
                        2022;3. <a href="https://doi.org/10.1158/2767-9764.CRC-22-0302" rel="noreferrer nofollow"
                                   target="_blank">https://doi.org/10.1158/2767-9764.CRC-22-0302</a>.
                    </li>
                    <li>Uhrik L. Study of structural changes within proteins important in tumour biology using
                        hydrogen-deuterium exchange coupled with mass spectrometry. Dissertation 2022.
                    </li>
                    <li>Urbanski L, Brugiolo M, Park S, Angarola BL, Leclair NK, Yurieva M, et al. MYC regulates a
                        pan-cancer network of co-expressed oncogenic splicing factors. Cell Reports 2022;41:111704. <a
                                href="https://doi.org/10.1016/j.celrep.2022.111704" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.celrep.2022.111704</a>.
                    </li>
                    <li>Vaddavalli PL, Schumacher B. The p53 network: cellular and systemic DNA damage responses in
                        cancer and aging. Trends in Genetics 2022;38:598–612. <a
                                href="https://doi.org/10.1016/j.tig.2022.02.010" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.tig.2022.02.010</a>.
                    </li>
                    <li>Van A, Kunkel M, Baffi T, et al. Protein kinase C fusion proteins are paradoxically
                        loss-of-function in cancer. J Biol Chem. 2021;296:100445.
                        https://doi.org/10.1016/j.jbc.2021.100445
                    </li>
                    <li>Vaysburd M. Identifying Epigenetic Signature of Breast Cancer with Machine Learning. arXiv;
                        2019.
                    </li>
                    <li>Vellichirammal NN, Albahrani A, Banwait JK, Mishra NK, Li Y, Roychoudhury S, et al. Pan-Cancer
                        Analysis Reveals the Diverse Landscape of Novel Sense and Antisense Fusion Transcripts.
                        Molecular Therapy - Nucleic Acids 2020;19:1379–98. <a
                                href="https://doi.org/10.1016/j.omtn.2020.01.023" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.omtn.2020.01.023</a>.
                    </li>
                    <li>Venkata Krishna Bharadwaj Parasaram. Designing Interoperable Microservices for Public-Private
                        Collaboration in U.S. Biomedical Research. ASRJETS-Journal 2025;103:290–312.
                    </li>
                    <li>Visci G, Tolomeo D, Agostini A, Traversa D, Macchia G, Storlazzi CT. CircRNAs and
                        Fusion-circRNAs in cancer: New players in an old game. vol. 75. Elsevier Inc.; 2020. <a
                                href="https://doi.org/10.1016/j.cellsig.2020.109747" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.cellsig.2020.109747</a>.
                    </li>
                    <li>Voropaeva EN, Orlov YL, Loginova AB, Seregina OB, Maksimov VN, Pospelova TI. Deregulation
                        mechanisms and therapeutic opportunities of p53-responsive microRNAs in diffuse large B-cell
                        lymphoma. PeerJ 2025;13:e18661. <a href="https://doi.org/10.7717/peerj.18661"
                                                           rel="noreferrer nofollow" target="_blank">https://doi.org/10.7717/peerj.18661</a>.
                    </li>
                    <li>Wakayama S, Ouchi K, Takahashi S, Yamada Y, Komatsu Y, Shimada K, et al. TP53 Gain-of-Function
                        Mutation is a Poor Prognostic Factor in High-Methylated Metastatic Colorectal Cancer. Clinical
                        Colorectal Cancer 2023. <a href="https://doi.org/10.1016/j.clcc.2023.06.001"
                                                   rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.clcc.2023.06.001</a>.
                    </li>
                    <li>Walter W, Iacobucci I, Meggendorfer M. Diagnosis of acute lymphoblastic leukaemia: an overview
                        of the current genomic classification, diagnostic approaches, and future directions.
                        Histopathology 2024:his.15338. <a href="https://doi.org/10.1111/his.15338"
                                                          rel="noreferrer nofollow" target="_blank">https://doi.org/10.1111/his.15338</a>.
                    </li>
                    <li>Walter W, Shahswar R, Stengel A, Meggendorfer M, Kern W, Haferlach T, et al. Clinical
                        application of whole transcriptome sequencing for the classification of patients with acute
                        lymphoblastic leukemia. BMC Cancer 2021;21:886. <a
                                href="https://doi.org/10.1186/s12885-021-08635-5" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1186/s12885-021-08635-5</a>.
                    </li>
                    <li>Wang C, Song H, Chen L, Li Q, Yang J, Hu XT, et al. Automatic Liver Segmentation Using
                        Multi-plane Integrated Fully Convolutional Neural Networks. 2018 IEEE International Conference
                        on Bioinformatics and Biomedicine (BIBM), 2018, p. 1–6. <a
                                href="https://doi.org/10.1109/BIBM.2018.8621257" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1109/BIBM.2018.8621257</a>.
                    </li>
                    <li>Wang H. Developing A Quantitative Systems Pharmacology Model for Clinical Trial Simulation in
                        Immuno-Oncology. Johns Hopkins University, 2024.
                    </li>
                    <li>Wang H, Guo M, Wei H, Chen Y. Targeting p53 pathways: mechanisms, structures, and advances in
                        therapy. Sig Transduct Target Ther 2023;8:1–35. <a
                                href="https://doi.org/10.1038/s41392-023-01347-1" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41392-023-01347-1</a>.
                    </li>
                    <li>Wang H, Zhao C, Santa-Maria CA, Emens LA, Popel AS. Dynamics of tumor-associated macrophages in
                        a quantitative systems pharmacology model of immunotherapy in triple-negative breast cancer.
                        iScience 2022;25:104702. <a href="https://doi.org/10.1016/j.isci.2022.104702"
                                                    rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.isci.2022.104702</a>.
                    </li>
                    <li>Wang L, Quine S, Frickenstein AN, Lee M, Yang W, Sheth VM, et al. Exploring and Analyzing the
                        Systemic Delivery Barriers for Nanoparticles. Advanced Functional Materials 2023;n/a:2308446. <a
                                href="https://doi.org/10.1002/adfm.202308446" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1002/adfm.202308446</a>.
                    </li>
                    <li>Wang L, Wang W, Beird H, Cheng X, Fang H, Tang G, et al. PPP1R7 Is a Novel Translocation Partner
                        of CBFB via t(2;16)(q37;q22) in Acute Myeloid Leukemia. Genes 2022.
                    </li>
                    <li>Wang N, Yuan L, Jing Y, Fan K, Jin H, Lv C, et al. Double minute chromosomes in acute myeloid
                        leukemia and myelodysplastic syndromes are associated with complex karyotype, monosomal
                        karyotype, TP53 deletion, and TP53 mutations. Leukemia & Lymphoma 2021;0:1–9. <a
                                href="https://doi.org/10.1080/10428194.2021.1919663" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1080/10428194.2021.1919663</a>.
                    </li>
                    <li>Wang T, Russo DP, Demokritou P, Jia X, Huang H, Yang X, et al. An Online Nanoinformatics
                        Platform Empowering Computational Modeling of Nanomaterials by Nanostructure Annotations and
                        Machine Learning Toolkits. Nano Lett 2024;24:10228–36. <a
                                href="https://doi.org/10.1021/acs.nanolett.4c02568" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1021/acs.nanolett.4c02568</a>.
                    </li>
                    <li>Wang T, Yang X, Zhu H. ViNAS-Pro: Online nanotoxicity data, modeling, and predictions. Machine
                        Learning and Artificial Intelligence in Toxicology and Environmental Health, Elsevier; 2026, p.
                        233–44. <a href="https://doi.org/10.1016/B978-0-443-30010-3.00011-8" rel="noreferrer nofollow"
                                   target="_blank">https://doi.org/10.1016/B978-0-443-30010-3.00011-8</a>.
                    </li>
                    <li>Wang X, Johnson V, Johnson L, Cook JR. RNA-Based next generation sequencing complements but does
                        not replace fluorescence in situ hybridization studies for the classification of aggressive
                        B-Cell lymphomas. Cancer Genetics 2021;252–253:43–7. <a
                                href="https://doi.org/10.1016/j.cancergen.2020.12.004" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.cancergen.2020.12.004</a>.
                    </li>
                    <li>Wang Y, Yu C, Pei G, Jia W, Li T, Li P. Dissolution of oncofusion transcription factor
                        condensates for cancer therapy. Nat Chem Biol 2023:1–12. <a
                                href="https://doi.org/10.1038/s41589-023-01376-5" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41589-023-01376-5</a>.
                    </li>
                    <li>Wang Z, Chu T, Christie MA, Marru S, Danko CG, Abeysinghe E, et al. Building a science gateway
                        for processing and modeling sequencing data via apache airavata. ACM International Conference
                        Proceeding Series, New York, NY, USA: Association for Computing Machinery; 2018, p. 1–7. <a
                                href="https://doi.org/10.1145/3219104.3219141" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1145/3219104.3219141</a>.
                    </li>
                    <li>Wang Z, Davidsen TM, Kuffel GR, Addepalli K, Bell A, Casas-Silva E, et al. NCI Cancer Research
                        Data Commons: Resources to Share Key Cancer Data. Cancer Research 2024. <a
                                href="https://doi.org/10.1158/0008-5472.CAN-23-2468" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1158/0008-5472.CAN-23-2468</a>.
                    </li>
                    <li>Wästerlid T, Cavelier L, Haferlach C, Konopleva M, Fröhling S, Östling P, et al. Application of
                        precision medicine in clinical routine in haematology—Challenges and opportunities. Journal of
                        Internal Medicine 2022;292:243–61. <a href="https://doi.org/10.1111/joim.13508"
                                                              rel="noreferrer nofollow" target="_blank">https://doi.org/10.1111/joim.13508</a>.
                    </li>
                    <li>Watson S, LaVigne CA, Xu L, Surdez D, Cyrta J, Calderon D, et al. VGLL2-NCOA2 leverages
                        developmental programs for pediatric sarcomagenesis. Cell Reports 2023;42:112013. <a
                                href="https://doi.org/10.1016/j.celrep.2023.112013" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.celrep.2023.112013</a>.
                    </li>
                    <li>Wen J, Chong ML. Acquired Ring Chromosomes in Solid Tumors. In: Li P, Liehr T, editors. Human
                        Ring Chromosomes, Cham: Springer International Publishing; 2024, p. 475–90. <a
                                href="https://doi.org/10.1007/978-3-031-47530-6_32" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1007/978-3-031-47530-6_32</a>.
                    </li>
                    <li>Wilhelm T, Said M, Naim V. DNA Replication Stress and Chromosomal Instability: Dangerous
                        Liaisons. Genes 2020;11. <a href="https://doi.org/10.3390/genes11060642"
                                                    rel="noreferrer nofollow" target="_blank">https://doi.org/10.3390/genes11060642</a>.
                    </li>
                    <li>Xia J, Yang G, Yu Q, Zhang R, Li G, Wang M, et al. Case report: Identification of a novel
                        TOR1AIP2::ETV6 transcript with FLT3-ITD mutation in acute myeloid leukemia progressed from
                        myelodysplastic syndrome. Front Oncol 2024;14:1466590. <a
                                href="https://doi.org/10.3389/fonc.2024.1466590" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.3389/fonc.2024.1466590</a>.
                    </li>
                    <li>Xiao X, Trinh TX, Gerelkhuu Z, Ha E, Yoon TH. Automated machine learning in nanotoxicity
                        assessment: A comparative study of predictive model performance. Computational and Structural
                        Biotechnology Journal 2024;25:9–19. <a href="https://doi.org/10.1016/j.csbj.2024.02.003"
                                                               rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.csbj.2024.02.003</a>.
                    </li>
                    <li>Xin Yu J, Hubbard-Lucey VM, Tang J. Immuno-oncology drug development goes global. vol. 18.
                        Nature Publishing Group; 2019. <a href="https://doi.org/10.1038/d41573-019-00167-9"
                                                          rel="noreferrer nofollow" target="_blank">https://doi.org/10.1038/d41573-019-00167-9</a>.
                    </li>
                    <li>Xue Z, Warren R, Gibb E, MacMillan D, Wong J, Chiu R, et al. Pan-cancer analysis reveals complex
                        tumor-specific alternative polyadenylation. BiorxivOrg 2017:160960. <a
                                href="https://doi.org/10.1101/160960" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1101/160960</a>.
                    </li>
                    <li>Xue Z, Warren RL, Gibb EA, MacMillan D, Wong J, Chiu R, et al. Recurrent tumor-specific
                        regulation of alternative polyadenylation of cancer-related genes. BMC Genomics 2018;19. <a
                                href="https://doi.org/10.1186/s12864-018-4903-7" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1186/s12864-018-4903-7</a>.
                    </li>
                    <li>Yamamoto K, Kawamoto S, Chijiki R, Watanabe M, Matsumoto S, Kitao A, et al. Biclonal diffuse
                        large B-cell lymphoma commonly characterized by partial trisomy 18q involving MALT1 and BCL2.
                        Internal Medicine 2022;advpub. <a href="https://doi.org/10.2169/internalmedicine.9711-22"
                                                          rel="noreferrer nofollow" target="_blank">https://doi.org/10.2169/internalmedicine.9711-22</a>.
                    </li>
                    <li>Yamamoto K, Kitao A, Watanabe M, Kanehira H, Joyce M, Hirakawa Y, et al. RUNX1 rearrangement in
                        mature B-cell acute lymphoblastic leukemia with non-L3 morphology. Journal of Clinical and
                        Experimental Hematopathology 2023a;advpub:23028. <a href="https://doi.org/10.3960/jslrt.23028"
                                                                            rel="noreferrer nofollow" target="_blank">https://doi.org/10.3960/jslrt.23028</a>.
                    </li>
                    <li>Yamamoto K, Kurata K, Higashime A, Sai S, Yakushijin K, Kawamoto S, et al. Non-inverted
                        Duplication of 11q, dup(11)(q11q24), in a Diffuse Large B Cell Lymphoma Without MYC
                        Rearrangement: Case Report. SN Comprehensive Clinical Medicine 2021a:1–8. <a
                                href="https://doi.org/10.1007/s42399-021-00866-8" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1007/s42399-021-00866-8</a>.
                    </li>
                    <li>Yamamoto K, Matsumoto H, Matsumoto S, Sakai R, Kitao A, Watanabe M, et al. Unexpected appearance
                        of KMT2A::MLLT10 fusion transcript in acute myeloid leukemia with t(5;11)(q31;q23.3). Cancer
                        Genetics 2023b;272–273:41–6. <a href="https://doi.org/10.1016/j.cancergen.2023.02.002"
                                                        rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.cancergen.2023.02.002</a>.
                    </li>
                    <li>Yamamoto K, Yakushijin K, Ito M, Goto H, Higashime A, Kajimoto K, et al. MYC amplification on
                        double minute chromosomes in plasma cell leukemia with double IGH/CCND1 fusion genes. Cancer
                        Genetics 2020;242:35–40. <a href="https://doi.org/10.1016/j.cancergen.2020.01.049"
                                                    rel="noreferrer nofollow" target="_blank">https://doi.org/10.1016/j.cancergen.2020.01.049</a>.
                    </li>
                    <li>Yamamoto K, Yakushijin K, Mizutani Y, Okuni-Watanabe M, Goto H, Higashime A, et al. Expression
                        of a novel type of KMT2A/EPS15 fusion transcript in FLT3 mutation-positive B-lymphoblastic
                        leukemia with t(1;11)(p32;q23). Cancer Genetics 2021b. <a
                                href="https://doi.org/10.1016/j.cancergen.2021.02.006" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.cancergen.2021.02.006</a>.
                    </li>
                    <li>Yan X, Yue T, Winkler DA, Yin Y, Zhu H, Jiang G, et al. Converting Nanotoxicity Data to
                        Information Using Artificial Intelligence and Simulation. Chem Rev 2023. <a
                                href="https://doi.org/10.1021/acs.chemrev.3c00070" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1021/acs.chemrev.3c00070</a>.
                    </li>
                    <li>Yang Y, Zhang G, Hu C, Luo W, Jiang H, Liu S, et al. The germline mutational landscape of
                        genitourinary cancers and its indication for prognosis and risk. BMC Urol 2022;22:196. <a
                                href="https://doi.org/10.1186/s12894-022-01141-1" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1186/s12894-022-01141-1</a>.
                    </li>
                    <li>Yoon SJ, Combs JA, Falzone A, Prieto-Farigua N, Caldwell S, Ackerman HD, et al. Comprehensive
                        Metabolic Tracing Reveals the Origin and Catabolism of Cysteine in Mammalian Tissues and Tumors.
                        Cancer Research 2023;83:1426–42. <a href="https://doi.org/10.1158/0008-5472.CAN-22-3000"
                                                            rel="noreferrer nofollow" target="_blank">https://doi.org/10.1158/0008-5472.CAN-22-3000</a>.
                    </li>
                    <li>You K, Chen T, Wang Y, Li P, Li T. Oncogenic Fusions with Aberrant Transcriptional Regulation.
                        Journal of Molecular Biology 2025;437:169268. <a
                                href="https://doi.org/10.1016/j.jmb.2025.169268" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1016/j.jmb.2025.169268</a>.
                    </li>
                    <li>Zainal-Abidin R-A, Khalid K. Nanotechnologies and Omics: A Way Forward. In: Al-Khayri J,
                        Alnaddaf LM, Jain SM, Penna S, editors. Innovative Methods in Horticultural Crop Improvement:
                        Molecular Tools, Nanotechnology and Artificial Intelligence, Cham: Springer International
                        Publishing; 2024, p. 3–23. <a href="https://doi.org/10.1007/978-3-031-61081-3_1"
                                                      rel="noreferrer nofollow" target="_blank">https://doi.org/10.1007/978-3-031-61081-3_1</a>.
                    </li>
                    <li>Zanetti M, Xian S, Dosset M, Carter H. The Unfolded Protein Response at the Tumor-Immune
                        Interface. Front Immunol 2022;13:823157. <a href="https://doi.org/10.3389/fimmu.2022.823157"
                                                                    rel="noreferrer nofollow" target="_blank">https://doi.org/10.3389/fimmu.2022.823157</a>.
                    </li>
                    <li>Zapata-Dongo R, Albornoz C, Llimpe Y. Acute Myeloid Leukemia with Minimal Differentiation with
                        translocation t(16;21)(p11;q22): Case report and review of the literature. 2020a. <a
                                href="https://doi.org/10.13140/RG.2.2.14551.60329" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.13140/RG.2.2.14551.60329</a>.
                    </li>
                    <li>Zapata-Dongo R, Paul O, Ramos S. Immunophenotype, cytogenetic and molecular characterization of
                        19 cases reported in Mitelman database with t(16;21)(q24;q22) in acute myeloid leukemia and its
                        relation to prognosis The MCL1 inhibitor S63845 in combination with the histone deacetylase
                        inhi. EuroLeuk 2020, 2020b. <a href="https://doi.org/10.13140/RG.2.2.34383.71845"
                                                       rel="noreferrer nofollow" target="_blank">https://doi.org/10.13140/RG.2.2.34383.71845</a>.
                    </li>
                    <li>Zedan HT, Ali FH, Zayed H. The spectrum of chromosomal translocations in the Arab world:
                        ethnic-specific chromosomal translocations and their relevance to diseases. Chromosoma 2022. <a
                                href="https://doi.org/10.1007/s00412-022-00775-2" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1007/s00412-022-00775-2</a>.
                    </li>
                    <li>Zhang B. Colorectal cancer predictive test using germ-line DNA data and multiple machine
                        learning methods. 2019.
                    </li>
                    <li>Zhang L, Abro B, Campbell A, Ding Y. TP53 mutations in myeloid neoplasms: implications for
                        accurate laboratory detection, diagnosis, and treatment. Laboratory Medicine 2024:lmae048. <a
                                href="https://doi.org/10.1093/labmed/lmae048" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1093/labmed/lmae048</a>.
                    </li>
                    <li>Zhang M, Yao X, Guan X, Jia C, Zhang R, Wang H, et al. Clinical relevance of BCOR internal
                        tandem duplication and TP53 aberration in clear cell sarcoma of the kidney. Human Pathology
                        2022. <a href="https://doi.org/10.1016/j.humpath.2022.12.007" rel="noreferrer nofollow"
                                 target="_blank">https://doi.org/10.1016/j.humpath.2022.12.007</a>.
                    </li>
                    <li>Zhou Y, Cui G, Xu H, Chun J, Yang D, Zhang Z, et al. Loss of TP53 cooperates with c-MET
                        overexpression to drive hepatocarcinogenesis. Cell Death Dis 2023;14:1–9. <a
                                href="https://doi.org/10.1038/s41419-023-05958-y" rel="noreferrer nofollow"
                                target="_blank">https://doi.org/10.1038/s41419-023-05958-y</a>.
                    </li>
                    <li>Ziegler E, Urban T, Brown D, Petts J, Pieper SD, Lewis R, et al. Open Health Imaging Foundation
                        Viewer: An Extensible Open-Source Framework for Building Web-Based Imaging Applications to
                        Support Cancer Research. JCO Clinical Cancer Informatics 2020:336–45. <a
                                href="https://doi.org/10.1200/CCI.19.00131" rel="noreferrer nofollow" target="_blank">https://doi.org/10.1200/CCI.19.00131</a>.
                    </li>
                    <li>Zou AYS. Adoption of Optical Genome Mapping in Clinical Cancer Cytogenetic Laboratory: A
                        Stepwise Approach – Journal of Clinical & Anatomic Pathology (JCAP). Journal of Clinical &
                        Anatomic Pathology 2021;6.
                    </li>
                    <li>Zupan A, Salapura V, Šekoranja D, Pižem J. Subcutaneous chondromyxoid fibroma with a novel
                        PNISR::GRM1 fusion—report of a primary soft tissue tumour without connection to an underlying
                        bone. Virchows Arch 2023. <a href="https://doi.org/10.1007/s00428-023-03519-4"
                                                     rel="noreferrer nofollow" target="_blank">https://doi.org/10.1007/s00428-023-03519-4</a>.
                    </li>
                </ol>
            </div>
        </div>
    </div>
</div>
    </div>

    <!-- Logout Modal -->
</main>

<!-- Warning Modal -->
<div class="modal fade" id="gov_warning" tabindex="-1" role="dialog" data-backdrop="static" aria-labelledby="gov_warning-header" aria-hidden="true">
    <div class="modal-dialog">
        <div class="modal-content">
            <div class="modal-header">
                <!-- <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button> -->
                <h4 class="modal-title" id="gov_warning-header">Warning</h4>
            </div>
            <div class="modal-body">
            <p>You are accessing a US Government web site which may contain information that must be protected under the
                US Privacy Act or other sensitive information and is intended for Government authorized use only.</p>
            <p>Unauthorized attempts to upload information, change information, or use of this web site may result in
                disciplinary action, civil, and/or criminal penalties. Unauthorized users of this website should have no
                expectation of privacy regarding any communications or data processed by this website.</p>
            <p>Anyone accessing this website expressly consents to monitoring of their actions and all communications or
                data transiting or stored on related to this website and is advised that if such monitoring reveals
                possible evidence of criminal activity, NIH may provide that evidence to law enforcement officials.</p>
            <p>The ISB-CGC system is authorized to be used by the public to view and analyze cancer data.</p>
            </div>
            <div class="modal-footer">
                <button type="button" class="btn btn-primary btn-warn">OK</button>
            </div>
        </div>
    </div>
</div>


<script>

    var warningSeen = true;
    var showWarning = false;
</script>




<script>
        fetch('../nav.html').then(res => res.text())
      .then(data => {
        document.getElementsByClassName('site-header')[0].innerHTML = data;
        document.getElementsByClassName('isblogo')[0].src="../static/img/isblogo.png"
      });

</script>


    <script src="/static/js/libs/require.js"></script>
    <script src="/static/js/base.js"></script>


</body>
</html>
